# Principles of Rational Drug Design
!!! Authors
    **Claude Cohen** (Synergix)

!!! Info
    This chapter is a general introduction on rational drug design. Basic concepts such as the lock-and-key and induced fit models are presented. Important approaches such as receptor-based and pharmacophore-based drug design are introduced.
*[pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[nce]: New Chemical Entity: a compound not previously described.


    **Number of Pages: 46 (Â±1 hours read)**

    Last Modified: January 2005

    Prerequisites: None
## Rational Drug Design

### Drug Design Basis: Molecular Recognition

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/drug-design-basis-molecular-recognition-e03c1173'><i class='fa fa-play' aria-hidden='true'></i></button>Life is based on molecular systems that are organized and regulated by precise recognition and discrimination process. Insights into biological systems show us that all the underlying molecular machinery operates in three-dimension. This is the basis of rational drug design.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/1_1_0_1.png){: loading=lazy }

### Lock-and-Key Model

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/lock-and-key-model-8021da13'><i class='fa fa-play' aria-hidden='true'></i></button>In 1890 Emil Fischer proposed a model called the "lock-and-key model" that explained how biological systems function. A substrate fits into the active site of a macromolecule, just like a key fits into a lock. Biological 'locks' have unique stereochemical features that are necessary to their function.
*[sar]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.


![](https://media.drugdesign.org/course/drug-design/lock_and_key.gif){: loading=lazy }

### Induced-Fit Model

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/induced-fit-model-787ee11d'><i class='fa fa-play' aria-hidden='true'></i></button>However, the lock-and-key model did not take into account the conformational changes that occur for both the ligand and the target macromolecule. An extension of this model was proposed by Daniel Koshland in 1958 and called the "induced-fit theory". This theory proposed that in the recognition process both ligand and target mutually adapt themselves by small conformational changes until an optimal fit is achieved.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/drug-design/induced_fit.gif){: loading=lazy }

### Rational Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/rational-drug-design-414a2fb9'><i class='fa fa-play' aria-hidden='true'></i></button>Rational drug design exploits the detailed recognition and discrimination features that are associated with the specific arrangement of the chemical groups in the active site of a target macromolecule. New molecules that can optimally interact with the protein can be conceived to block or trigger a specific given biological action.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/drug-design/1_3_0_1.jpg){: loading=lazy }

### Rational Drug Design Process

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/rational-drug-design-process-6a633b62'><i class='fa fa-play' aria-hidden='true'></i></button>Rational drug design proceeds in three steps. It begins with the design of compounds that conform to specific requirements. Then the molecules are synthesized and tested. Further rounds allow the refinement of a given scaffold and its optimization.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/rational_drug_design_process.gif){: loading=lazy }

### Receptor-Based Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/receptor-based-drug-design-32b06c8f'><i class='fa fa-play' aria-hidden='true'></i></button>If the three-dimensional structure of the target protein is known, this information can be directly exploited for the retrieval and design of new ligands. This approach is called "structure-based", "receptor-based", "target-based" or "direct" drug design.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ligands]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/1_5_0_1.png){: loading=lazy }

### Pharmacophore-Based Drug Design
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/pharmacophore-based-drug-design-d9153dbd'><i class='fa fa-play' aria-hidden='true'></i></button>When the three-dimensional structure of the target protein is not available, one can exploit the information provided by known active molecules. The interaction with the target protein remains the central issue, however the design is made indirectly. This approach is called "pharmacophore-based", "ligand-based" or "indirect" drug design.
*[pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/1_6_0_1.png){: loading=lazy }

## Pharmacophore-Based Design
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.

### Pharmacophore-Based Drug Design Approach
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/pharmacophore-based-drug-design-approach-d635f8eb'><i class='fa fa-play' aria-hidden='true'></i></button>Pharmacophore-based drug design starts with the knowledge of active/inactive compounds. The design is based on the stereochemical and physicochemical features of the known reference compounds.
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/drug-design/2_1_0_1.png){: loading=lazy }

### Similarity Concepts and Molecular Mimicry
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/similarity-concepts-molecular-mimicry-8fcf8795'><i class='fa fa-play' aria-hidden='true'></i></button>In indirect design, the analysis of the active and inactive molecules gives a "feeling" of how structural variations can change biological properties and allows one to generate hypotheses about the interactions of the ligands with the receptor. For example, one might deduce: " This group is necessary; this substituent suppresses the biological activity in this series etc." This strategy consists of positioning the 3D relative location of the structural elements recognized in the active molecules as necessary in the structure of new chemical entities. This strategy is called "molecular mimicry".
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ligands]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[sar]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.


![](https://media.drugdesign.org/course/drug-design/2_2_0_1.png){: loading=lazy }

### Examples of Molecular Mimicry

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/examples-molecular-mimicry-a045a2cd'><i class='fa fa-play' aria-hidden='true'></i></button>The following pages will illustrate some examples of molecular mimicry between chemically unrelated compounds such as hormones, cofactors (ATP), steroids, peptides, biogenic amines, carbohydrates, and fatty acids.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/example_mimicry.gif){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-molecular-mimicry' class='nglviewer-container' data-molecule-id='molecular-mimicry' style='width: 750px; height: 450px;' data-initialized='false'></div>

#### ATP

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/atp-9b1c5541'><i class='fa fa-play' aria-hidden='true'></i></button>The following view illustrates a molecule that is mimicking ATP, the cofactor of all the protein kinases (inhibitors of protein kinases have great potential in various diseases including cancer, psoriasis, restenosis, rheumatoid arthritis).
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[nce]: New Chemical Entity: a compound not previously described.


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/atp.gif){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-atp' class='nglviewer-container' data-molecule-id='atp' style='width: 750px; height: 450px;' data-initialized='false'></div>

#### Dopamine

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/dopamine-151d9aed'><i class='fa fa-play' aria-hidden='true'></i></button>The following view illustrates a molecule that is mimicking the structure of dopamine, a biogenic amine (dopamine antagonists are implicated in CNS diseases).
*[antagonist]: A molecule that blocks the activation of a receptor.
*[antagonists]: A molecule that blocks the activation of a receptor.
*[agonist]: A chemical substance capable of activating a given receptor.
*[agonists]: A chemical substance capable of activating a given receptor.


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/dopamine.gif){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-dopamine' class='nglviewer-container' data-molecule-id='dopamine' style='width: 750px; height: 450px;' data-initialized='false'></div>

#### Histamine

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/histamine-3f09011e'><i class='fa fa-play' aria-hidden='true'></i></button>The following view illustrates a molecule that is mimicking histamine (useful as anti-allergic, gastric or cardio vascular systems therapies).


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/histamine.gif){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-histamine' class='nglviewer-container' data-molecule-id='histamine' style='width: 750px; height: 450px;' data-initialized='false'></div>

#### Estradiol

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/estradiol-f85e23b0'><i class='fa fa-play' aria-hidden='true'></i></button>The following view illustrates a molecule that is mimicking estradiol. Estrogen modulators have potential applications in various therapies (e.g. cancer, cardiovascular and osteoporosis).
*[nce]: New Chemical Entity: a compound not previously described.


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/estradiol.gif){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-estradiol' class='nglviewer-container' data-molecule-id='estradiol' style='width: 750px; height: 450px;' data-initialized='false'></div>

### Peptidomimetics
*[Peptidomimetics]: Compounds that are peptide mimetics. They have non-peptide structures in part or in full, and mimic the the pharmacological activity of peptides.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/peptidomimetics-9b207532'><i class='fa fa-play' aria-hidden='true'></i></button>When molecular mimicry is applied to peptides, the discipline is called "peptidomimetics". This type of approach is necessary in drug research when dealing with a peptide molecule that cannot be developed as a drug. This topic is presented with some details in the chapter "peptidomimetics".
*[peptidomimetics]: Compounds that are peptide mimetics. They have non-peptide structures in part or in full, and mimic the the pharmacological activity of peptides.
*[sar]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/2_4_0_1.png){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-peptidomimetics' class='nglviewer-container' data-molecule-id='peptidomimetics' style='width: 750px; height: 450px;' data-initialized='false'></div>

### Strengths of Pharmacophore-Based Drug Design
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/strengths-pharmacophore-based-drug-design-0fae164e'><i class='fa fa-play' aria-hidden='true'></i></button>Despite our ignorance of anchorage points and the exact interactions that occur with the receptor site, pharmacophore-based drug design has the advantage of guiding the discovery process in projects where the receptor is not known. Moreover, the chances of success are greater when we start with something that looks like a solution (active molecule) than by starting from the problem (protein structure).
*[pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/drug-design/2_5_0_1.png){: loading=lazy }

## Receptor-Based Design

### Design by Direct Interaction with Receptor Sites

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/design-by-direct-interaction-receptor-sites-5cecfae5'><i class='fa fa-play' aria-hidden='true'></i></button>When the three-dimensional structure of the target protein is known, the receptor-based design exploits the recognition and discrimination capabilities of the receptor site to create direct interactions between the designed molecule and atoms or functional groups of the target protein. The macromolecular target can be a protein or a nucleic acid.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/3_1_0_1.png){: loading=lazy }

### Exploiting the Receptor Recognition Concepts
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/exploiting-receptor-recognition-concepts-d686fb26'><i class='fa fa-play' aria-hidden='true'></i></button>A crystallographic structure reveals the essential structural features of the active site. It is then used to generate new ideas how to create new prototype molecules. Specific intermolecular interactions can be incorporated in the ligand to strengthen its binding to the active site of the target protein.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/drug-design/receptor.gif){: loading=lazy }

### Initial Data in Receptor-Based Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/initial-data-receptor-based-drug-design-3ae328e7'><i class='fa fa-play' aria-hidden='true'></i></button>The best starting point in receptor-based drug design is to have the X-ray structure of the target protein complexed with a ligand. If this is not available, one can generate such information by modeling techniques from the primary sequence of the target protein and by homology with the X-ray structures of homologous proteins.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/drug-design/3_3_0_1.jpg){: loading=lazy }

### Strengths of Receptor Based Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/strengths-receptor-based-drug-design-734c27d8'><i class='fa fa-play' aria-hidden='true'></i></button>Structure-based design has the advantage of providing a "visual" framework for the direct design of new prototypes. It allows one to utilize detailed 3D features of the active site by introducing appropriate functionalities in the designed ligand being considered. The modeler can rapidly assess the validity of a possible solution and measure the progress achieved in the course of successive design attempts.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/drug-design/3_4_0_1.jpg){: loading=lazy }

## Integration in a Global Perspective

### Typical Projects

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/typical-projects-7f6ec015'><i class='fa fa-play' aria-hidden='true'></i></button>A typical project is rarely a "pure" pharmacophore-based or a "pure" receptor-based project. Information comes from many sources and disciplines that provide a sequence and/or molecules and/or crystallographic structures. There are many intermediate stages in the development of the project, which are constantly changing according to newly generated information.
*[pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/drug-design/4_1_0_1.png){: loading=lazy }
### Exploit the Two Methods, Independently

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/exploit-two-methods-independently-b714531f'><i class='fa fa-play' aria-hidden='true'></i></button>When information is available for both the target protein and active molecules, the two approaches can be developed independently. In the first case the design will concentrate on the binding to the 3D structure of the protein, and in the second case it will be based on the structures of the reference active molecules.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/drug-design/4_2_0_1.png){: loading=lazy }

### Synergy Between the Two Approaches
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/synergy-between-two-approaches-1490efd5'><i class='fa fa-play' aria-hidden='true'></i></button>The ideal situation is to find a way of combining the two approaches; the synergy that is created can substantially accelerate the discovery process. The project can then be intelligently conducted by generating molecules with one approach and by subjecting them to analyses with the second approach. In such a situation, the synergy can be realized.


![](https://media.drugdesign.org/course/drug-design/4_3_1_1.jpg){: loading=lazy }

### Example of Synergy Between the Two Approaches
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/example-synergy-between-two-approaches-36e824a8'><i class='fa fa-play' aria-hidden='true'></i></button>For example, when a promising docked molecule is designed and modeled with favorable interactions with the target protein, it is compared to the active structures. Likewise, when an interesting mimic of an active compound is considered, it is docked into the protein to see if the two approaches lead to convergent conclusions.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/synergy.gif){: loading=lazy }

### Good Binding Models, the Synergy Condition
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/good-binding-models-synergy-condition-e87e514e'><i class='fa fa-play' aria-hidden='true'></i></button>In order to take advantage of the synergy between direct and indirect design, one has to find the link between the two approaches, which consists of positioning correctly the active molecules into the active site of the target protein. This is called "binding mode" of the ligand.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/4_4_1_1.png){: loading=lazy }

#### Ideal Situation

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/ideal-situation-49176e2d'><i class='fa fa-play' aria-hidden='true'></i></button>The ideal situation is to have X-ray structures of complexes between the active compounds and the target protein. However even when the X-ray data is not available, one can predict and construct a model of the binding mode of a ligand by using modeling techniques.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/ideal_situation.gif){: loading=lazy }

#### Example 1

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/example-1-6d78bcba'><i class='fa fa-play' aria-hidden='true'></i></button>A new insulin receptor modulator was discovered and shown to bind to the receptor in a non-competitive way. The X-ray structure of the binding domain of the free receptor was available and attempts to predict the binding mode were unsuccessful. In such a case, the synergy between the two sources of data was not feasible thus the molecular modeler could only exploit the two routes independently.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/no_link.png){: loading=lazy }

#### Example 2

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/example-2-251403a9'><i class='fa fa-play' aria-hidden='true'></i></button>A potent insulin receptor modulator was discovered that competed with the natural substrate, however the compound was toxic. The X-ray structure of the complex between the target protein and its subtrate is available. Preliminary analyses showed structural similarities between the substrate and the synthetic molecule. The molecular modeler could exploit the two approaches independently but also try to establish a synergistic link between them.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/easy.png){: loading=lazy }

### Integration in a Global Perspective

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/integration-global-perspective-3bafabaa'><i class='fa fa-play' aria-hidden='true'></i></button>To summarize the three important components of any drug discovery project are the following: the disease, the reference molecules and the target protein considered.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/4_5_1_1.png){: loading=lazy }
#### Pharmacophore-Based Drug Design
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/pharmacophore-based-drug-design-35304c4b'><i class='fa fa-play' aria-hidden='true'></i></button>Pharmacophore-based drug design exploits the link between reference molecules and the disease at the molecular level.
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/4_5_2_1.png){: loading=lazy }
#### Receptor-Based Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/receptor-based-drug-design-c030cfc1'><i class='fa fa-play' aria-hidden='true'></i></button>Receptor-based drug design exploits the link between the target (protein, gene etc...) and the disease at the molecular level.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/4_5_3_1.png){: loading=lazy }
#### Integrated Global Approach

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/integrated-global-approach-a830b6bb'><i class='fa fa-play' aria-hidden='true'></i></button>The integrated global approach aims at the identification of a structural model rationalizing the biological activities of the known molecules on the basis of their interactions with the 3D structure of the target protein.


![](https://media.drugdesign.org/course/drug-design/4_5_4_1.png){: loading=lazy }
## Challenge of the Genomics Era

### The Genomic Era

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/genomic-era-6af6d0a6'><i class='fa fa-play' aria-hidden='true'></i></button>Genomics is the study of genes and their function. It allows the linking of specific genes to specific diseases and provides insights into our understanding of the molecular mechanisms of diseases. It is the driving force behind the discovery and development of drugs based on the understanding of diseases at the molecular level.


![](https://media.drugdesign.org/course/drug-design/5_1_0_1.jpg){: loading=lazy }

### A New Challenge in Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/new-challenge-drug-design-681de2c7'><i class='fa fa-play' aria-hidden='true'></i></button>The challenge for rational drug design in our genomic era is to take advantage of knowledge generated by genomics and to convert it into useful therapeutic agents. The challenge is twofold because of the changes that are introduced by the massive information that is generated by both genomics and proteomics.


![](https://media.drugdesign.org/course/drug-design/5_2_0_1.png){: loading=lazy }

## Typical Projects

### Typical Pharmacophore-Based Project
*[Pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/typical-pharmacophore-based-project-ee3f37c2'><i class='fa fa-play' aria-hidden='true'></i></button>The chemical formulas of a series of biologically active molecules are known and act with the same mechanism of action. Based on these structures the aim of the project is to create novel proprietary structures with improved profiles. This is a typical starting point of a pharmacophore-based approach aiming at the creation of new lead molecules mimicking the structures of a known reference series.
*[pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/drug-design/6_1_1_1.png){: loading=lazy }
### Design Based on 3D Mimicry

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/design-based-on-3d-mimicry-38a488c8'><i class='fa fa-play' aria-hidden='true'></i></button>A molecule can be designed that mimics the structural features common to the set of molecules considered.


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/mimic.gif){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-mimicry' class='nglviewer-container' data-molecule-id='mimicry' style='width: 750px; height: 450px;' data-initialized='false'></div>

### Typical Receptor-Based Project

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/typical-receptor-based-project-c5585e65'><i class='fa fa-play' aria-hidden='true'></i></button>The development of a drug candidate that inhibits a cancer-related protein was stopped due to insufficient selectivity. The X-ray data of the complex of the molecule with the protein concerned is available. This is a possible starting point for a receptor-based project aiming at the discovery of potent and selective molecules.
*[nce]: New Chemical Entity: a compound not previously described.


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/receptor_based.png){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-ligand-protein' class='nglviewer-container' data-molecule-id='ligand-protein' style='width: 750px; height: 450px;' data-initialized='false'></div>

### Design Based in Making Favorable 3D Interactions

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/design-based-making-favorable-3d-interactions-a87e6759'><i class='fa fa-play' aria-hidden='true'></i></button>A molecule can be designed that has optimal interactions with the 3D structure of the target protein.


=== "Gif"
    ![](https://media.drugdesign.org/course/drug-design/receptor_based2.gif){: loading=lazy }

=== "3D"
    <div id='nglviewer-container-interaction' class='nglviewer-container' data-molecule-id='interaction' style='width: 750px; height: 450px;' data-initialized='false'></div>

### Typical Genomic Project

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/typical-genomic-project-43036ea5'><i class='fa fa-play' aria-hidden='true'></i></button>A gene having an adenine instead of a thymine has been shown to be at the origin of a particular hereditary disease resulting in obesity. This impaired gene encodes an aspartic protease which represents the first step of the cascade causing obesity. This is a typical starting point for a project derived from genomics and aiming at the discovery of anti-obesity non-peptidic agents that would be inhibitors of this protein.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/6_3_0_1.jpg){: loading=lazy }

## Perspectives

### Drug Discovery of the 1970's 

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/drug-discovery-1970s-c5a58ce2'><i class='fa fa-play' aria-hidden='true'></i></button>Since in the 1970's, no X-ray structure of a disease-relevant biological target was available, drug research was essentially based on the structures of active molecules (e.g. penicillin, steroids, peptides, alkaloids etc...).
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/drug-design/7_1_1_1.png){: loading=lazy }
### Drug Discovery of the 1980's

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/drug-discovery-1980s-04d84217'><i class='fa fa-play' aria-hidden='true'></i></button>In the 1980's, protein homology models were constructed to compensate for the lack of X-ray data in order to provide data for direct drug design of new molecules. Receptor-based drug design started to "take-off".


![](https://media.drugdesign.org/course/drug-design/7_1_2_1.png){: loading=lazy }
### Drug Discovery of the 1990's

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/drug-discovery-1990s-6e2ab087'><i class='fa fa-play' aria-hidden='true'></i></button>In the 1990's, recombinant DNA and cloning technologies greatly helped to the isolation and crystallization of many key biological targets. Many rational drug design projects were developed and based on the X-ray structure of a target protein (e.g. HIV-1, Renin, and Thrombin).


![](https://media.drugdesign.org/course/drug-design/7_1_3_1.png){: loading=lazy }
### The Present Situation

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/present-situation-3a73787a'><i class='fa fa-play' aria-hidden='true'></i></button>Presently the structural 3D features of many target proteins are available, increasing the number of projects where the receptor-based approach is used. However combinatorial chemistry and high through-put screening generate data that can be exploited in a pharmacophore-based perspective.
*[pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/7_1_4_1.png){: loading=lazy }
### Initial Skepticism Towards Rational Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/initial-skepticism-towards-rational-drug-design-e6d60208'><i class='fa fa-play' aria-hidden='true'></i></button>Rational drug design has generated initial skepticism. The following published cartoon, illustrates the ironic attitude that it generated. The picture suggests that the rational approach intended to establish a correlation between the shape of the drop of water and the shape of the lock of the door!
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/7_2_0_1.png){: loading=lazy }

### Success Stories in Rational Drug Design

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/success-stories-rational-drug-design-bfc65335'><i class='fa fa-play' aria-hidden='true'></i></button>Meanwhile, rational drug design has become a mature discipline and has proven to efficiently contribute to the drug discovery process. Today the number of success stories is so huge that it is impossible to review them in a single article. The examples presented in this course are all illustrations of how rational drug design has contributed to real achievements in drug discovery.


![](https://media.drugdesign.org/course/drug-design/7_3_0_1.png){: loading=lazy }

### Future Perspectives

<button  class='playb' onclick='playAudio(this)' data-mp3-name='drug-design/future-perspectives-5ed3c670'><i class='fa fa-play' aria-hidden='true'></i></button>Rational drug design is only at the beginning of its development and is progressing rapidly. Each of the disciplines mentioned is making its own spectacular progress. This indicates that new concepts and approaches will be soon in demand from computerized molecular design technologies. Rational drug design will be able to provide drug discovery with an entirely new dimension. The starting point and the data accumulated in a given project will be treated in a global perspective and will generate an unexpected synergy between projects that may appear unrelated.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/drug-design/7_4_0_1.png){: loading=lazy }



Copyright Â© 2024 drugdesign.org
