# Structural Bioinformatics
!!! Authors
    **Eran Eyal** (Pittsburgh University, USA and the Weizmann Institute of Science, Rehovot, Israel) , **Elie Cohen** (Synergix) and **Claude Cohen** (Synergix)
*[nce]: New Chemical Entity: a compound not previously described.

!!! Info
    Structural Bioinformatics is a discipline that unites experimental and theoretical fields that all have the 3D structures of macromolecules in common. It aims at understanding how biological systems work at the molecular level. In this chapter we present the organization of macromolecules, their 3D determination, visualization and structural classification together with the principles of molecular recognition, structure-based drug design, and applications of physical and chemical principles with algorithms for the simulation of macromolecular structures.


    **Number of Pages: 135 (Â±3 hours read)**

    Last Modified: June 2008

    Prerequisites: None
## Introduction to Structural Bioinformatics

### Challenges in the Post Genomic Era

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/challenges-post-genomic-era-465ffb96'><i class='fa fa-play' aria-hidden='true'></i></button>The completion of the human genome project, which successfully determined the sequence of human DNA, was the first step towards a comprehensive understanding of life at the molecular level. This heralded the dawn of a new era and assigned researchers and scientists the challenge of analyzing, deciphering and understanding the genetic origin of human disease. One of the key issues is the characterization and function of proteins with a view to using the findings in the discovery of novel and effective therapeutic treatments.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/intro.png){: loading=lazy }
### The Informational Chaos

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/informational-chaos-4a6e31c0'><i class='fa fa-play' aria-hidden='true'></i></button>As a consequence of advances in the fields of molecular biology, genomics, proteomics, structural biology and drug discovery, the post-genomic era has been characterized by the uncontrolled explosion of information. This includes vast amounts of genetic data, identification of numerous potential targets, large amounts of biological information, huge amounts of solved crystal structures, large quantities of SAR data from HTS, etc. New strategies are absolutely crucial if we are to face the challenges of the post-genomic era.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[SAR]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.
*[structural biology]: Structural Biology is a new field of modern biology dedicated to the determination of the 3D structure of biological proteins to give insight to the function of these macromolecules in the living cell and to exploit this knowledge for the design of new drugs.


![](https://media.drugdesign.org/course/structural-bioinformatics/chaos_informational.png){: loading=lazy }
### Integration through Computational Science
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/integration-through-computational-science-ef53e994'><i class='fa fa-play' aria-hidden='true'></i></button>To be able to handle the vast volumes of data generated by new high-throughput technologies, new high-throughput computational disciplines have emerged to face this challenge: computational biology to solve problems in biology; bioinformatics to solve problems in genomics; computational chemistry to solve problems in chemistry and cheminformatics to solve problems in drug discovery.
*[computational chemistry]: A discipline based on mathematical models for the calculation or the simulation of molecular properties.


![](https://media.drugdesign.org/course/structural-bioinformatics/need_of_methods.png){: loading=lazy }

??? Abstract "wikipedia"
    [Computational Biology](http://en.wikipedia.org/wiki/Computational_biology) 
    
    [Computational Chemistry](http://en.wikipedia.org/wiki/Computational_chemistry) 
    
    [Cheminformatics](http://en.wikipedia.org/wiki/Cheminformatics) 
    
    [Bioinformatics](http://en.wikipedia.org/wiki/Bioinformatics) 
    
### Structural Bioinformatics

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/structural-bioinformatics-05f290db'><i class='fa fa-play' aria-hidden='true'></i></button>Structural Bioinformatics is an interdisciplinary field that deals with the three dimensional structures of biomolecules. It attempts to model and discover the basic principles underlying biological machinery at the molecular level. It is based on the belief that 3D structural information of a biological system is the core to understanding its mechanism of action and function. Structural bioinformatics combines applications of physical and chemical principles with algorithms from computational science.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/sbi1.png){: loading=lazy }


??? Abstract "wikipedia"
    [Structural bioinformatics](http://en.wikipedia.org/wiki/Structural_bioinformatics) 
    
### Grouping Fields into One Discipline

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/grouping-fields-into-one-discipline-ae4ad6ae'><i class='fa fa-play' aria-hidden='true'></i></button>For the lay person, it can be confusing to figure out where structural bioinformatics fits into other disciplines. Although it emerged as a new discipline in the 2000s, it was in fact born in the early 1950s as a result of the crystallographic determinations of hemoglobin and myoglobin. Structural bioinformatics has now become an interdisciplinary field with scientists that are all involved in the 3D structures of macromolecules.


![](https://media.drugdesign.org/course/structural-bioinformatics/assembly_of_fields.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Medicinal Chemist](http://en.wikipedia.org/wiki/Medicinal_chemistry) 
    
    [Crystallographer](http://en.wikipedia.org/wiki/X-ray_crystallography) 
    
    [Computational Chemist](http://en.wikipedia.org/wiki/Computational_chemistry) 
    
    [NMR Spectroscopist](http://en.wikipedia.org/wiki/Nuclear_magnetic_resonance) 
    
    [Biochemist](http://en.wikipedia.org/wiki/Biochemist) 
    
    [Molecular Modeler](http://en.wikipedia.org/wiki/Molecular_modelling) 
    
    [Enzymologist](http://en.wikipedia.org/wiki/Enzyme) 
    
### 3D Basis of Structural Bioinformatics

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/3d-basis-structural-bioinformatics-f0d736e9'><i class='fa fa-play' aria-hidden='true'></i></button>Structural Bioinformatics is based on the protein structure/function paradigm, which states that the function of a macromolecule is dictated by its 3D-structure. As a consequence, if we want to predict the function of a protein we need to start from its 3D representation and try to decode the hidden information embedded in this structure. The challenges for our post genomic era towards a systematic understanding of life are: (1) to obtain the 3D structures of all the proteins encoded in the genomes and (2) to decode their function based on their structure.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/methods_for_3d.png){: loading=lazy }
### The Structural Genomics Effort

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/structural-genomics-effort-bc59bf5e'><i class='fa fa-play' aria-hidden='true'></i></button>"Structural Genomics" aims at obtaining the 3D structures of all the proteins encoded in the genome in a systematic way. Since protein folding remains an unsolved problem, theoretical approaches to translating sequences into 3D protein structures are unfeasible; and scientists have to rely mainly on experimental determinations. A number of worldwide structural genomics programs are in progress; their common aim is to make large-scale, high-throughput protein structure determinations.
*[nce]: New Chemical Entity: a compound not previously described.
*[protein folding]: A spontaneous process directed by the physical and chemical properties of the amino acid sequence that dictate the folding of a protein into a particular conformation.


![](https://media.drugdesign.org/course/structural-bioinformatics/plan_steps.png){: loading=lazy }

??? Abstract "articles"
    **Making decisions for structural genomics** 
    Rodrigues A, Hubbard R.E 
    Briefings in Bioinformatics 
    4 2003 10.1093/bib/4.2.150 
    

??? Abstract "wikipedia"
    [Structural Genomics](http://en.wikipedia.org/wiki/Structural_genomics) 
    
#### The Protein Structure Initiative

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/protein-structure-initiative-8d107d77'><i class='fa fa-play' aria-hidden='true'></i></button>The National Institute of General Medical Sciences (NIGMS/NIH) founded in 2000 one of the first structural genomics programs: the "Protein Structure Initiative" (PSI). The goal of the project is to reveal the exact 3D structure of all proteins of the genomes by experimental methods or computational approaches. The PSI has started to create a large collection of protein structures determined by high-throughput techniques using X-ray crystallography and NMR spectroscopy.
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/exp_pred_method2.png){: loading=lazy }

??? Abstract "wikipedia"
    [Structural genomics](http://en.wikipedia.org/wiki/Structural_genomics) 
    
#### Strategy of the Protein Structure Initiative

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/strategy-protein-structure-initiative-74f07f35'><i class='fa fa-play' aria-hidden='true'></i></button>Structure determination is a long and expensive process. To achieve its goal the NIGMS classifies proteins into families and concentrates its effort on experimentally solving the 3D structure of only one member of each family. All the other members of a family can be modeled using homology modeling techniques.
*[homology modeling]: A method for predicting the 3D structure of a protein, based on its amino acid sequence and the 3D-structure of analog protein(s).
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/clustering.png){: loading=lazy }

??? Abstract "wikipedia"
    [Homology Modeling - Virginia Tech](http://en.wikipedia.org/wiki/Homology_modeling) 
    
#### The Structural Genomics Consortium

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/structural-genomics-consortium-ca909362'><i class='fa fa-play' aria-hidden='true'></i></button>Due to the formidable challenges posed by the structural genomics effort, other pragmatic initiatives were taken based on a prioritization of the targets to be achieved. For example, the "Structural Genomics Consortium" was created in 2004 with the aim of solving the three dimensional structures of proteins known to be of medical relevance. The Consortium has defined a list of 2000 disease-relevant proteins. In 4 years of activities the consortium has solved 386 proteins and plans to solve 50% of the proteins on the list by 2011.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/sgc_consortium.png){: loading=lazy }
#### Global Planning of Structural Genomics

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/global-planning-structural-genomics-0e34ab60'><i class='fa fa-play' aria-hidden='true'></i></button>The key difference between structural genomics and traditional structural biology lies in the global strategy adopted by structural genomics to cover the entire area of protein structures. The strategy consists of selecting targets with high informational content, avoiding redundancy and systematically spanning the entire molecular space. This global planning has paved the way to the determination of 3D structures of all the genome proteins.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[structural biology]: Structural Biology is a new field of modern biology dedicated to the determination of the 3D structure of biological proteins to give insight to the function of these macromolecules in the living cell and to exploit this knowledge for the design of new drugs.


=== "Planification"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/global_plan.png){: loading=lazy }
=== "3D structures of all the genomes proteins"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/global_plan_predict.gif){: loading=lazy }

#### The Impact of Structural Genomics

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/impact-structural-genomics-3b74a443'><i class='fa fa-play' aria-hidden='true'></i></button>About 20% of the new structures deposited in the PDB are from SG, but the real contribution of SG is in new structures from families that do not have 3D representatives. In 2005 only 36% of the Pfam families (2736 of 7677) had a 3D representative. In their study, Chandonia and Brenner analyzed the evolution of the contribution of SG, and showed that in 2005 about 50% of the total number of first representatives of protein families had been solved by SG organizations.
*[PDB]: The "Protein Data Bank". A worldwide repository source of 3D structures of proteins obtained by X-ray crystallography or NMR studies.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[Pfam]: The Pfam database contains information about protein domains and families.


![](https://media.drugdesign.org/course/structural-bioinformatics/impact.png){: loading=lazy }

??? Abstract "articles"
    **The impact of structural genomics: expectations and outcomes.** 
    Chandonia, J. M. and Brenner, S. E 
    Science 
    311 2006 
           
    

??? Abstract "wikipedia"
    [Pfam](http://en.wikipedia.org/wiki/Pfam) 
    

???+ Abstract "Media"
    Diagram adapted from the article of Chandonia and Brenner (2006) 
    [
          ](
          )  
    
### The Relationship between Structure and Function
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/relationship-between-structure-function-a64269d1'><i class='fa fa-play' aria-hidden='true'></i></button>The comparison between sequence similarities and the 3D structures of proteins has become a major source of discoveries and predictions in biology. Much of the information deduced from the structure may provide important clues to guide researchers towards function. Many properties that are not apparent in the sequence (e.g. spatial proximity of amino acids) are apparent in the 3D structure.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/20_1nbf_oxianion.gif){: loading=lazy }

??? PDB
    **PDB entry: 1nbf** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1nbf) 
    
#### Example of a Structure-Function Relationship

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-structure-function-relationship-92b8d5bb'><i class='fa fa-play' aria-hidden='true'></i></button>Subtilisin and chymotrypsin are two proteins that take on completely different folds despite the fact that the two proteins have identical functions: they break peptide bonds. Their common function is due to the presence of three residues (Asp, His, Ser) forming a catalytic triad that has exactly the same geometric arrangement, although located at different positions in the primary sequence.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "catalytic triad"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/structure_function_1.png){: loading=lazy }
=== "subtilisin"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/alt1_1bh6.jpg){: loading=lazy }

=== "chymotrypsin"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/alt1_4cha.jpg){: loading=lazy }

??? PDB
    **PDB entry: 4CHA** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=4cha) 
    
    **PDB entry: 1bh6** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1bh6) 
    

??? Abstract "book"
    **** 
    Petsko G.A. and Ringe D. 
    Protein Structure and Function 
    Sinauer Associates  2004   
    

??? Abstract "wikipedia"
    [Catalytic triad](http://en.wikipedia.org/wiki/Catalytic_triad) 
    
#### Learning from Evolution

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/learning-from-evolution-b75ee400'><i class='fa fa-play' aria-hidden='true'></i></button>Evolution provides an important key to function by taking advantage of the similarity principle to generate information by comparative analyses. Typically, homolog proteins (descended from a common ancestor) have significant sequence similarity and similar three-dimensional structures and functions. However, structures show higher levels of conservation than sequences and the comparison of 3D structures is more useful in revealing functional relationships that are hidden at the primary sequence level.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/evolution_3d.png){: loading=lazy }

??? PDB
    **PDB entry: 1ecd** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ecd) 
    
    **PDB entry: 1mbd** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1mbd) 
    

???+ Abstract "Media"
    This picture was made using the MATRAS server 
    [3D Matras Alignment](http://biunit.aist-nara.ac.jp/matras/matras_pair.html)  
    
#### Learning from Structural Folds

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/learning-from-structural-folds-28a47eb4'><i class='fa fa-play' aria-hidden='true'></i></button>The presence of a local structural motif can provide information about the functional role of a protein. In the case of the protein shown below, the presence of a "helix-turn-helix" motif in the folded structure of the protein hints at the ability of this protein to bind to DNA.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/motif_dna.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Helix-turn-helix](http://en.wikipedia.org/wiki/Helix-turn-helix) 
    
#### Learning from Molecular Shape

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/learning-from-molecular-shape-6441edaa'><i class='fa fa-play' aria-hidden='true'></i></button>Biomolecules do not see and recognize each other as a set of atoms and bonds; rather they perceive shapes carrying complex forces. Analysis of molecular surfaces of proteins with their associated physico-chemical properties (such as electrostatic potentials etc...), can provide detailed information that is useful in understanding the properties and the function of these proteins. An important field of structural bioinformatics called 'molecular docking' aims at predicting computationally the association of two molecular surfaces.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Docking"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/docking.png){: loading=lazy }

=== "Electrostatic Potential"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/electrostatic_potential.png){: loading=lazy }

???+ Abstract "Media"
    This docking picture shows the binding of ubiquitin aldehyde with UBP 
    [Docking](http://www.pdb.org/pdb/cgi/explore.cgi?pdbId=1NBF)  
    
    The electrostatic potential was generated using the DeepView Swiss-pdb Viewer 
    [Electrostatic potential picture](
          )  
    
#### Example of Knowledge Derived from 3D Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-knowledge-derived-from-3d-structure-8e740bac'><i class='fa fa-play' aria-hidden='true'></i></button>One of the first examples of 3D knowledge that led to a better understanding of life processes at the molecular level was the Watson and Crick DNA double helix model deduced from the X-ray photographs of DNA strands made by Rosalind Franklin in the early 1950s. This was a major step forward in charting many biological processes such as translation, regulation and replication that are carried out through the interaction of other molecules with DNA.
*[f DNA ]: Food and Drug Administration. Agency in the USA responsible for safety regulations.


![](https://media.drugdesign.org/course/structural-bioinformatics/rosalin.png){: loading=lazy }
### Is Structure Sufficient to Predict Function?

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/is-structure-sufficient-to-predict-function-94b532e9'><i class='fa fa-play' aria-hidden='true'></i></button>Proteins are not static; to fulfill their function they undergo a wide range of motion and conformational rearrangements. The knowledge of a static structure may not be sufficient to understand its biological function. An important component of structural bioinformatics is the development of experimental and computational methods (such as molecular dynamics) that sheds light on the dynamic behavior of biological systems. These methods simulate the way biomolecules behave, specifically recognize each other, interact and trigger biological events at the molecular level.
*[molecular dynamics]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system


![](https://media.drugdesign.org/course/structural-bioinformatics/why_3d_important.png){: loading=lazy }
### Exploiting Knowledge to Design New Drugs

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/exploiting-knowledge-to-design-new-drugs-b167c9c6'><i class='fa fa-play' aria-hidden='true'></i></button>The main application of structural bioinformatics is the discovery of novel therapeutic agents. By knowing the 3D architecture of a macromolecular system, rational drug design exploits the detailed features of recognition and discrimination that are associated with the specific arrangement of the chemical groups of a target macromolecule. New molecules are designed to interact optimally with the macromolecule to block or trigger a specific biological action.


![](https://media.drugdesign.org/course/structural-bioinformatics/1hck_1fin_agouron.png){: loading=lazy }


??? Abstract "wikipedia"
    [Drug Design](http://en.wikipedia.org/wiki/Drug_design) 
    
### Bridge between Genomics and Drug Discovery
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/bridge-between-genomics-drug-discovery-2087ba9a'><i class='fa fa-play' aria-hidden='true'></i></button>Structural bioinformatics is the bridging discipline between bioinformatics (the genomics world) and cheminformatics (the drug discovery world). It is the key to the "gene-to-drug" dream and answers questions such as: "how is known genetic information converted into a therapeutically useful agent?" The principle of the approach is to translate both genomic and small molecule knowledge into 3D structural data that can then be exploited in the structure-based drug design world.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/bridge.png){: loading=lazy }

??? Abstract "wikipedia"
    [Drug Discovery](http://en.wikipedia.org/wiki/Drug_discovery) 
    
### Tools Developed by Structural Bioinformatics

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/tools-developed-by-structural-bioinformatics-7b291f46'><i class='fa fa-play' aria-hidden='true'></i></button>To achieve its goals structural bioinformatics focuses on the development of tools for: (1) encoding and visualizing biomolecules, (2) predicting the 3D structure of biomolecules, (3) storing, retrieving and searching in structural databases, (4) analyzing structural data, (5) performing data-mining, (6) calculating binding scores and (7) simulating molecular processes.


![](https://media.drugdesign.org/course/structural-bioinformatics/method.png){: loading=lazy }
## Architecture of Biomolecules

### Biomolecules in the Cell

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/biomolecules-cell-1b935582'><i class='fa fa-play' aria-hidden='true'></i></button>In this section we describe the architecture of the two most important biomolecules present in our cells: nucleic acids and proteins.


![](https://media.drugdesign.org/course/structural-bioinformatics/biomolecules.png){: loading=lazy }
### DNA/RNA Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/dnarna-structure-b039f754'><i class='fa fa-play' aria-hidden='true'></i></button>Nucleic acids are essential materials found in all living organisms. Their main function is to maintain and transmit the genetic code; as such, they are the information carriers in the cells. This information is stored in the form of tremendously long polymer chains. Although the information they carry is one-dimensional, it is essential to understand the 3D structure of nucleic acids. This structure dictates their organization, functions and interactions with proteins.


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_intro.png){: loading=lazy }

??? Abstract "wikipedia"
    [DNA](http://en.wikipedia.org/wiki/DNA) 
    
#### DNA is the Genetic Material

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/dna-is-genetic-material-9280e2db'><i class='fa fa-play' aria-hidden='true'></i></button>DNA (Deoxyribo Nucleic Acid) is the genetic material of the vast majority of living cells. The language of the genetic code contains only 4 letters (A,T,G,C). Information about protein synthesis, which is the main machinery in the cells, is carried as shown in the table. Each triplet of nucleotides ("codon") encodes one amino acid. In the figure, click on one amino acid to visualize its corresponding codon.


![](https://media.drugdesign.org/course/structural-bioinformatics/genetic_code2.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Genetic Code](http://en.wikipedia.org/wiki/Genetic_code) 
    
#### DNA Variability

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/dna-variability-01ebfd29'><i class='fa fa-play' aria-hidden='true'></i></button>The genetic code is extremely long (3x10<sup>9</sup> characters in humans), which is enough for the phenotypic variability seen in nature. Because of its length, the DNA molecules must be carefully packed and preserved. Many proteins in the cell are involved in these processes.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_package.gif){: loading=lazy }


???+ Abstract "Media"
    This picture was made using the VMD and the Virtual DNA Viewer Extension 
    [VMD](http://www.ks.uiuc.edu/Research/vmd/plugins/)  
    
#### Importance of the DNA 3D Structure
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/importance-dna-3d-structure-2014582e'><i class='fa fa-play' aria-hidden='true'></i></button>In order to understand how the DNA fulfills its function, it is necessary to understand its structure. In 1952, Hershey showed that DNA is indeed genetic material. In 1953 Watson and Crick, based on Rosalind Franklin's research, modeled the double helix structure of DNA. Their model accounted perfectly for the role of DNA as the genetic code and its ability to replicate. DNA folds into specific three-dimensional conformations, which are partially determined by the sequence of nucleotides.
*[f DNA.]: Food and Drug Administration. Agency in the USA responsible for safety regulations.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[sar]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.


![](https://media.drugdesign.org/course/structural-bioinformatics/1bna.gif){: loading=lazy }

![](https://media.drugdesign.org/course/structural-bioinformatics/snap_sbi_02_05.jpg){: loading=lazy }

??? PDB
    **PDB entry: 1bna** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1bna) 
    

??? Abstract "wikipedia"
    [Alfred Hershey](http://en.wikipedia.org/wiki/Alfred_Hershey) 
    
    [Rosalind Franklin](http://en.wikipedia.org/wiki/Rosalind_Franklin) 
    
    [James D. Watson](http://en.wikipedia.org/wiki/James_Watson) 
    
    [Francis Crick](http://en.wikipedia.org/wiki/Francis_Crick) 
    
#### The Building Blocks

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/building-blocks-7c0cfd29'><i class='fa fa-play' aria-hidden='true'></i></button>Three types of chemicals make up the building blocks for nucleic acids: an aromatic base, a sugar ring and a phosphate group. The sugar and the phosphate constitute the non-specific backbone of the DNA. The base constitutes the specific part which actually holds the information and accounts for most of the interactions with other molecules.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_component.png){: loading=lazy }
##### Base

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/base-fc28d3d9'><i class='fa fa-play' aria-hidden='true'></i></button>The group that gives each nucleic acid unit its specificity is the organic base. DNA contains two purine bases (adenine and guanine) and two pyrimidine bases (cytosine and thymine). Purine bases are made of two planar aromatic rings, whereas pyrimidine bases contain only one aromatic ring. In RNA the thymine base is replaced by uracil. Polar atoms in the ring or attached to the ring are capable of creating hydrogen bonds with polar atoms of other bases.


![](https://media.drugdesign.org/course/structural-bioinformatics/base.png){: loading=lazy }
##### Sugar

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/sugar-29f8bbf9'><i class='fa fa-play' aria-hidden='true'></i></button>The second chemical group in nucleic acids is the sugar. The sugars in DNA and RNA are pentoses (5 carbons). The ring of the sugar is flexible and non planar (in contrast to the rings of the base), and can adopt several conformations.


=== "2D"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_ribose.png){: loading=lazy }
=== "3D"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_ribose.png){: loading=lazy }


??? Abstract "wikipedia"
    [Carbohydrate](http://en.wikipedia.org/wiki/Carbohydrate) 
    
##### Phosphate

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/phosphate-1bdbe468'><i class='fa fa-play' aria-hidden='true'></i></button>The inorganic acid H3PO4 (phosphoric acid) gives the nucleic acids an overall net negative charge. A basic DNA building block unit (nucleotide) contains three attached phosphate groups. However, in the polymerization of DNA two phosphates are released and one is incorporated into the DNA. Most of the interactions between the DNA and proteins which are not specific to the DNA sequence are with the phosphate groups.
*[f DNA ]: Food and Drug Administration. Agency in the USA responsible for safety regulations.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_mono_tri_p.png){: loading=lazy }

??? Abstract "wikipedia"
    [Phosphate](http://en.wikipedia.org/wiki/Phosphate) 
    
#### Putting the Building Blocks Together

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/putting-building-blocks-together-5f075397'><i class='fa fa-play' aria-hidden='true'></i></button>The three groups composing each unit of nucleic acids are connected by covalent bonds. The phosphate group is connected to carbon number 5 of the sugar. The sugar is connected to the base by a covalent single bond between carbon number 1 of the pentose ring and a nitrogen atom on the aromatic ring of the base. The assembly of these three groups defines a nucleotide. Nucleosides consist of the base and the sugar without the phosphate.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_together.png){: loading=lazy }
#### Nomenclature of Nucleotides and Nucleosides

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/nomenclature-nucleotides-nucleosides-771197a0'><i class='fa fa-play' aria-hidden='true'></i></button>Nucleotides are the monomers of nucleic acids. They may have one, two or three phosphate groups; the sugar can be ribose or deoxy-ribose and the base can be a purine or a pyrimidine base. Use the cursor to select a base, a sugar and a phosphate to visualize the corresponding nucleotide.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_nucleotide2.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Nucleoside](http://en.wikipedia.org/wiki/Nucleoside) 
    
    [Nucleotide](http://en.wikipedia.org/wiki/Nucleotide) 
    
#### Nucleotides of Nucleic Acids

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/nucleotides-nucleic-acids-b5ebf43b'><i class='fa fa-play' aria-hidden='true'></i></button>Mono-phosphate nucleotides are the structural units of nucleic acids. In DNA the sugar is always a deoxy-ribose. The nomenclature of the different nucleotides and nucleosides of DNA is shown in the table below.
*[f DNA ]: Food and Drug Administration. Agency in the USA responsible for safety regulations.


=== "Nucleotide"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_nucleotide.png){: loading=lazy }
=== "Nucleoside"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_nucleoside.png){: loading=lazy }
#### The Double Helix Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/double-helix-structure-cbfe690c'><i class='fa fa-play' aria-hidden='true'></i></button>In the early 1950s, Watson and Crick suggested that DNA was made of two helical strands of polynucleotides, one going up and the other going down. Their model proved to be correct and explained how, during cell division, the two strands separate and promote the assembly of two new strands identical to the ones existing before. This double helical structure enabled us to understand how the genetic information is maintained and how it is replicated during cell division.


![](https://media.drugdesign.org/course/structural-bioinformatics/helix_his.png){: loading=lazy }

??? Abstract "wikipedia"
    [Double helix](http://en.wikipedia.org/wiki/Double_helix) 
    
##### DNA Helices are Antiparallel

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/dna-helices-are-antiparallel-cf410be2'><i class='fa fa-play' aria-hidden='true'></i></button>Each strand of DNA has directionality. In other words, one edge is not identical to the other and there is no symmetry along the strand. The molecule has two different edges, termed the 3-prime edge and the 5-prime edge. The DNA molecule is arranged such that the two strands are lying next to each other but in opposite directions, such that the 3' edge of one strand is adjacent to the 5' of the other strand and vice versa. A phosphate group is always found in the 5' edge and hydroxyl group is found in the 3'. Polymerization occurs at the 3' edge.
*[f DNA ]: Food and Drug Administration. Agency in the USA responsible for safety regulations.


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_antiparallel.png){: loading=lazy }
##### Hydrogen Bonding Pattern

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/hydrogen-bonding-pattern-6c49d29e'><i class='fa fa-play' aria-hidden='true'></i></button>The two strands of the DNA are connected by hydrogen bonds. These forces are non-covalent interactions but are relatively strong and keep the two DNA strands in a tight association. Each nucleotide is capable of creating favorable hydrogen bond patterns with one other nucleotide. Adenine and Thymine interact with two hydrogen bonds. Cytosine and Guanine interact with three hydrogen bonds and therefore have a stronger association.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "2D pattern"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_hb.png){: loading=lazy }
=== "3D"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_hb.gif){: loading=lazy }

##### Aromatic Base Stacking

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/aromatic-base-stacking-7ed833ec'><i class='fa fa-play' aria-hidden='true'></i></button>Another important interaction that stabilizes the DNA structure is termed stacking and refers to the interaction between adjacent parallel aromatic bases. This interaction results from sharing the electron clouds of the aromatic rings. The interaction is slightly stronger in purines, which possess a double ring.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_stacking.png){: loading=lazy }

??? Abstract "wikipedia"
    [Stacking](http://en.wikipedia.org/wiki/Stacking_%28chemistry%29) 
    
##### Major and Minor Grooves

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/major-minor-grooves-30e14e12'><i class='fa fa-play' aria-hidden='true'></i></button>The two DNA strands are not straight; they form a helix. Due to its nature, this helix contains grooves which are preferred sites for proteins and other molecules to interact with the DNA. One significant groove in which the bases are highly exposed is termed the major groove. Most of the protein interactions occur at this site. The minor groove is located on the other side of the bases. It also serves for interactions with proteins, but less often.


=== "Grooves"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_groove.png){: loading=lazy }
=== "Rotatable Surface"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_groove.png){: loading=lazy }

#### DNA forms
*[rms]: Root Mean Square: the square root of the arithmetic average of the square's set of values.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/dna-forms-7788b379'><i class='fa fa-play' aria-hidden='true'></i></button>There are several forms of DNA double helices. The most common is the B-DNA. In every B-DNA cycle there are 10 base pairs; the distance between successive bases along the molecule axis is about 3.4 &#8491;. In dehydrated environments, the DNA may appear as A-DNA. In every A-DNA cycle there are 11 base pairs so the molecule is more condensed, and accordingly the diameter of A-DNA is greater than that of B-DNA. Another uncommon form of DNA is the Z-DNA. The helix has an opposite rotation, composed of C and G nucleotides and is less symmetrical.
*[f DNA ]: Food and Drug Administration. Agency in the USA responsible for safety regulations.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[rms]: Root Mean Square: the square root of the arithmetic average of the square's set of values.


=== "B-DNA and A-DNA"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_a_b_forms.png){: loading=lazy }
=== "Z-DNA"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_z_form.gif){: loading=lazy }

??? PDB
    **PDB entry: 1bna (B-FORM)** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1bna) 
    
    **PDB entry: 1ana (A-FORM)** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ana) 
    
    **PDB entry: 131d (Z-FORM)** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=131d) 
    
#### G-Quadruplex Conformation

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/g-quadruplex-conformation-c42f7fd1'><i class='fa fa-play' aria-hidden='true'></i></button>In addition to the duplex DNA conformations, certain DNA sequences can fold into structures that are four-stranded. In particular Guanine-rich nucleic acid sequences are capable of adopting this type of organization, which is called G-quadruplex. The G-quadruplex structure is stabilized by hydrogen bonds between the edges of the bases and chelation with a metal (e.g. potassium) located in the center. Below the 3D and 2D structure of a G-quadruplex is illustrated.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "3D"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/1kf1.gif){: loading=lazy }

=== "2D"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/quadruplex_2d.png){: loading=lazy }
??? PDB
    **PDB entry: 1kf1** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1kf1) 
    

??? Abstract "book"
    **** 
    
           
    Quadruplex Nucleic Acids 
    Royal Society of Chemistry, Cambridge  2007   
    

??? Abstract "wikipedia"
    [DNA (Quadruplex structures)](http://en.wikipedia.org/wiki/DNA#Quadruplex_structures) 
    
    [G-Quadruplex](http://en.wikipedia.org/wiki/G-quadruplex) 
    
#### DNA versus RNA

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/dna-versus-rna-fee298d7'><i class='fa fa-play' aria-hidden='true'></i></button>There are several important differences between DNA and RNA molecules. (1) The sugar of RNA is always a ribose instead of a deoxyribose (with an extra hydroxyl group at carbon number 2 of the sugar), (2) the RNA molecule does not contain thymine bases; instead they are replaced by uracil, and (3) the DNA molecule normally appears as a double strand, while the RNA molecule appears as a single strand.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/dna_rna_diff.png){: loading=lazy }
##### 3D Folds of RNA

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/3d-folds-rna-3ca11ffa'><i class='fa fa-play' aria-hidden='true'></i></button>Due to its single strand structure, the RNA molecule is less stable than DNA. Base pairing might occur between bases along the single strand giving rise to various three dimensional folds. Note that in the structure below (hairpin fold) intra-molecular hydrogen bond interactions are formed between bases of the same single strand.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Ribbon"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/hairpin1.gif){: loading=lazy }
=== "Stick"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/hairpin2.gif){: loading=lazy }

??? Abstract "wikipedia"
    [Hairpin (stem-loop)](http://en.wikipedia.org/wiki/Stem-loop) 
    
### Protein Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/protein-structure-e4b7b1e5'><i class='fa fa-play' aria-hidden='true'></i></button>Basic principles in protein structure are illustrated in this topic, which covers their amino acid building blocks and their folded, complex 3D architecture.


![](https://media.drugdesign.org/course/structural-bioinformatics/prot_structure.png){: loading=lazy }

??? PDB
    **PDB entry: 2TOH** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2TOH) 
    

??? Abstract "wikipedia"
    [Protein structure](http://en.wikipedia.org/wiki/Protein_structure) 
    

???+ Abstract "Media"
    This picture was made using KiNG (Kinetic Image, Next Generation) 
    [KiNG Home page](http://kinemage.biochem.duke.edu)  
    
#### Proteins are Fundamental to Life

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/proteins-are-fundamental-to-life-16e7d895'><i class='fa fa-play' aria-hidden='true'></i></button>Proteins form the very basis of life. They are the most abundant macromolecules in living cells. Thousands of different proteins can be found in a single cell, and proteins have been identified in all parts of all cells. Proteins affect virtually every property that characterizes a living organism.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/cell.png){: loading=lazy }
#### Structural Diversity of Proteins

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/structural-diversity-proteins-a1ede2d2'><i class='fa fa-play' aria-hidden='true'></i></button>The great diversity of the functional properties of proteins is directly related to their subtle three-dimensional structures. Every protein has a unique 3D structure that determines its function.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/structural_diversity.png){: loading=lazy }
#### Importance of Protein 3D Structures
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/importance-protein-3d-structures-9d136686'><i class='fa fa-play' aria-hidden='true'></i></button>The formation of a protein in its biologically active form requires the folding of the protein into a precise three-dimensional structure. Changes in this structure (for example by denaturation) are consistently associated with loss of function of that protein.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/denaturation.png){: loading=lazy }
#### Chemical Nature of Proteins

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/chemical-nature-proteins-305cc615'><i class='fa fa-play' aria-hidden='true'></i></button>All proteins are linear polymers made up of various combinations of 20 amino acids, and are composed mainly of C, H, O and N atoms. The amino acid sequence that defines the protein is determined by the transcription of the genetic code. Remarkably, a cell can produce proteins with different properties and activities by assembling the same 20 amino acids in many different combinations and sequences.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/chemical_nature.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Amino acid](http://en.wikipedia.org/wiki/Amino_acid) 
    
#### Challenges in Understanding Protein Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/challenges-understanding-protein-structure-63cb14fc'><i class='fa fa-play' aria-hidden='true'></i></button>Despite the exciting advances in our understanding of protein folding in the last decade, it remains an unresolved enigma. The prediction of the 3D structure of a protein from sequence data alone is a formidable challenge.
*[nce]: New Chemical Entity: a compound not previously described.
*[protein folding]: A spontaneous process directed by the physical and chemical properties of the amino acid sequence that dictate the folding of a protein into a particular conformation.


![](https://media.drugdesign.org/course/structural-bioinformatics/understand_prot_str.png){: loading=lazy }

??? Abstract "wikipedia"
    [Protein folding](http://en.wikipedia.org/wiki/Protein_folding) 
    

???+ Abstract "Media"
    This picture was released into the public domain 
    [Wikipedia](http://en.wikipedia.org/wiki/Image:Protein_folding.png)  
    
#### Protein Structure Complexity

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/protein-structure-complexity-07eb5aff'><i class='fa fa-play' aria-hidden='true'></i></button>The most striking feature of the folded conformation of a protein is its complexity, revealing the organization and the relationships of thousands of atoms in three-dimensional space. We will ease our way into the confusing jumble of atomic coordinates, distances, torsions angles and interactions with a short overview of protein architecture.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/protein_complexity.png){: loading=lazy }

??? PDB
    **PDB entry: 1MBN** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MBN) 
    
#### The Four Levels of Protein Architecture

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/four-levels-protein-architecture-954afa44'><i class='fa fa-play' aria-hidden='true'></i></button>Protein structure can be decomposed into different levels that follow precise rules and contribute to its overall shape. This is called the protein architecture, which can be split in a hierarchy of four levels of structural complexity. These correspond to the primary, secondary, tertiary and quaternary levels of protein structure.


![](https://media.drugdesign.org/course/structural-bioinformatics/architecture.png){: loading=lazy }
##### Primary Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/primary-structure-96f5c3c4'><i class='fa fa-play' aria-hidden='true'></i></button>The first and most fundamental level of a protein is its primary structure, which describes the atomic connectivities of the protein. The primary structure is defined by the amino acid sequence and may include cysteine bridges that are formed during cross-linking.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/primary.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Primary Structure](http://en.wikipedia.org/wiki/Primary_structure) 
    
##### Secondary Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/secondary-structure-11f81727'><i class='fa fa-play' aria-hidden='true'></i></button>The next level of protein structure, the secondary structure, includes local regular conformations of the polypeptide chain. These are dictated by the hydrogen bonds of the biopolymer creating specific secondary structures that are energetically favored in the conformational &Phi;-&Psi; space of the protein. The two major elements of the secondary structure are the alpha-helix and the &beta;-sheet.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Secondary Structure"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/architecture.gif){: loading=lazy }

=== "Ramachandran Plot"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/prot_level2_b.png){: loading=lazy }

??? Abstract "wikipedia"
    [Secondary structure](http://en.wikipedia.org/wiki/Secondary_structure) 
    
##### Tertiary Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/tertiary-structure-415ddd9b'><i class='fa fa-play' aria-hidden='true'></i></button>The tertiary structure of a protein is the overall 3D architecture of the folded polypeptide chain, which also includes the assembly of the various secondary structure elements in 3D.


![](https://media.drugdesign.org/course/structural-bioinformatics/tertiary.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Tertiary Structure](http://en.wikipedia.org/wiki/Tertiary_structure) 
    
##### Quaternary Structure

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/quaternary-structure-e814e23a'><i class='fa fa-play' aria-hidden='true'></i></button>Finally, proteins consisting of more than one polypeptide chain display a quaternary structure, which refers to the spatial relationship between the individual polypeptide chains (subunits).
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/quaternary.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Quaternary structure](http://en.wikipedia.org/wiki/Quaternary_structure) 
    
## Biomolecular Properties

### Protein Flexibility and Motion

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/protein-flexibility-motion-2f7cd6ca'><i class='fa fa-play' aria-hidden='true'></i></button>Proteins were considered to be static objects for a long time. This static view did not take into account the true dynamic nature of bio-molecules that are in constant movement, making motions of diverse amplitudes and time scales, from the level of individual atoms up to the level of structural domains. These motions are crucial to the biological functions of proteins.


![](https://media.drugdesign.org/course/structural-bioinformatics/dynamic_a1.gif){: loading=lazy }


??? Abstract "articles"
    **Einfluss der Configuration auf die Wirkung der Enzyme** 
    Fischer E. 
    Ber. Dt. Chem. Ges. 
    27 1894  
    

??? Abstract "wikipedia"
    [The lock and key model](http://en.wikipedia.org/wiki/Lock-and-key_model_%28enzyme%29) 
    
#### Importance of Dynamic Motions in Biological Processes
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/importance-dynamic-motions-biological-processes-14f25998'><i class='fa fa-play' aria-hidden='true'></i></button>Molecular motions can have two distinct roles. First they allow biomolecules to interact and adapt to other biomolecules (induced-fit), and second they dictate the biochemical or mechanical function of macromolecules. This function is generally only one small part of a larger molecular cell program. Examples of induced-fit and functional roles are given in the next pages.


![](https://media.drugdesign.org/course/structural-bioinformatics/dynamics_role.png){: loading=lazy }

??? Abstract "wikipedia"
    [Molecular Machinery](http://en.wikipedia.org/wiki/Molecular_machinery) 
    
##### Example of Function: ATP Synthase

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-function-atp-synthase-d4c777b1'><i class='fa fa-play' aria-hidden='true'></i></button>ATP Synthase is the universal carrier of energy in the cell. The enzyme catalyses the synthesis of ATP from ADP. ATP biosynthesis results in the production of a complex molecular machine composed of two connected rotary motors, where each motor is powered by a different fuel. Animation of ATP Synthase can be seen by activating the link appearing in the "additional information" panel (on the left side) which illustrates the importance of molecular motions in biological processes.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/atp_synthase.png){: loading=lazy }

??? Abstract "wikipedia"
    [ATP synthase](http://en.wikipedia.org/wiki/ATP_Synthase) 
    

???+ Abstract "Media"
    From Wikipedia, the free encyclopedia, This picture has been released into the public domain by the copyright holder 
    [ATPsynthase Picture](http://en.wikipedia.org/wiki/Image:ATPsynthase_labelled.png)  
    
##### Example of Function: DNA Biosynthesis

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-function-dna-biosynthesis-e9ea59b4'><i class='fa fa-play' aria-hidden='true'></i></button>Another example illustrating the importance of protein motion is DNA polymerase, an enzyme that catalyses DNA biosynthesis. This is a highly complex dynamic process where DNA polymerase assists DNA replication. In the figure below the structure of the DNA polymerase is shown both at a 3D and a schematic level.
*[nce]: New Chemical Entity: a compound not previously described.


=== "DNA Polymerase in 3D"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_replication.png){: loading=lazy }
=== "Schematic Representation"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/dna_poly.png){: loading=lazy }
??? PDB
    **PDB entry: 2avt** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2avt) 
    

??? Abstract "wikipedia"
    [DNA Polymerase](http://en.wikipedia.org/wiki/DNA_polymerase) 
    

???+ Abstract "Media"
    The schematic representation image has been released into the public domain by its author 
    [DNA replication picture](http://en.wikipedia.org/wiki/Image:DNA_replication.svg)  
    
##### Example of Function: Molecular Switch

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-function-molecular-switch-e4e5f019'><i class='fa fa-play' aria-hidden='true'></i></button>The differences in the arginine kinase shown below in two different conformational states result from the motion between four rigid groups of non-sequential residues, a change that acts as a molecular switch that controls cellular processes: switch "on" (blue) and switch "off" (red). The origin of the conformational changes induced in the different parts of the enzyme are the outcomes of the interactions with both ATP and the substrate of the kinase.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Switch 'On' and 'Off'"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/ex-arginine.png){: loading=lazy }
=== "Superimposition"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/ex_arginine2.png){: loading=lazy }

??? Abstract "articles"
    **Induced fit in guanidino kinases-comparison of substrate-free and transition state analog structures of arginine kinase** 
    Yousef M.S. et al. 
    Protein Science 
    12 2003 10.1110/ps.0226303 
    
??? PDB
    **PDB entry: 1M15** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M15) 
    
    **PDB entry: 1M80** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M80) 
    
##### Example of Induced-Fit: RNA-Protein Recognition

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-induced-fit-rna-protein-recognition-c0b038e3'><i class='fa fa-play' aria-hidden='true'></i></button>Studies of the interactions between RNA and proteins reveal that the formation of almost every RNA-Protein complex involves conformational changes in the protein, the RNA, or both ("mutually induced-fit"). In the example below the complex formed between RNA and the U1A protein is shown as well as the structures of the free molecules. In this example both molecules undergo important conformational changes.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/ex-rna.png){: loading=lazy }

??? Abstract "articles"
    **Induced fit in RNA-protein recognition** 
    Williamson J.R. 
    Nature Structural Biology 
    7 2000 10.1038/79575 
    
##### Example of Induced-Fit: Ubiquitous Proteins
*[Ubiquitous]: Play different roles, at the same time.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-induced-fit-ubiquitous-proteins-622ae99a'><i class='fa fa-play' aria-hidden='true'></i></button>Many proteins are ubiquitous and can bind to different protein targets. This is the case for Calmodulin, which undergoes conformational changes upon binding to calcium and can then bind to a number of specific proteins to elicit a specific response.
*[ubiquitous]: Play different roles, at the same time.


![](https://media.drugdesign.org/course/structural-bioinformatics/alt1_calmo.jpg){: loading=lazy }

![](https://media.drugdesign.org/course/structural-bioinformatics/Calmodulin_600.png){: loading=lazy }

![](https://media.drugdesign.org/course/structural-bioinformatics/snap_sbi_03_07.jpg){: loading=lazy }

??? PDB
    **PDB entry: 1k93 (CHAIN D)- orange** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1k93) 
    
    **PDB entry: 1cm1 (CHAIN A)** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1cm1) 
    
#### Types of Molecular Motions

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/types-molecular-motions-225da6f4'><i class='fa fa-play' aria-hidden='true'></i></button>Protein flexibility has different levels. (1) Individual atoms exercise very high frequency vibrations but are usually not very important for the function of the protein. (2) The rotation around single bonds are important in terms of the function of the protein because they lead to collective motions of several residues such as loop changes or side chain rotations; they accompany conformational changes such as those occurring in the vicinity of binding sites. (3) Important conformational changes associated with the relative motions of large structural units; they are extremely important for protein function.
*[rms]: Root Mean Square: the square root of the arithmetic average of the square's set of values.


![](https://media.drugdesign.org/course/structural-bioinformatics/mol_motion_type.gif){: loading=lazy }

#### Time Scale of Protein Motion

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/time-scale-protein-motion-b38fb391'><i class='fa fa-play' aria-hidden='true'></i></button>Illustrated below are the time scales for the different types of motions previously mentioned. Protein motions occur at time scales in the range of pico to milli-seconds. A motion starts with the side chains that change their rotameric state. It continues to loop movements, hinge bending and subunit displacements, respectively.


![](https://media.drugdesign.org/course/structural-bioinformatics/time_scale.png){: loading=lazy }
#### Methods to Study Protein Motions

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/methods-to-study-protein-motions-05682102'><i class='fa fa-play' aria-hidden='true'></i></button>There are many methods to study protein motions ranging from experimental techniques to simulation methods with computers. A brief outline of these methods is given in the following pages.


![](https://media.drugdesign.org/course/structural-bioinformatics/method_to_study_protein_motion.png){: loading=lazy }
##### Experimental Techniques to Study Protein Motions

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/experimental-techniques-to-study-protein-motions-a8e79be1'><i class='fa fa-play' aria-hidden='true'></i></button>Experimental 3D structure determination methods provide some indications as to macromolecular dynamics. X-ray crystallography assigns a value for the fluctuation of each atom. This parameter is known as the B-factor (temperature-factor) that expresses the mobility/uncertainty in the position of the corresponding atom. NMR spectroscopy, with measures of relaxation times and order-parameters tells us about the dynamics of the molecules. Other methods such as Mossbauer spectroscopy, neutron scattering and hydrogen exchange can also be used for studying protein mobility. Another way of acquiring information about the dynamics of a protein is to superimpose the different X-ray structures solved for a protein (i.e. complexes made with several inhibitors).
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[nce]: New Chemical Entity: a compound not previously described.
*[molecular dynamics]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system


![](https://media.drugdesign.org/course/structural-bioinformatics/experimental_motion.png){: loading=lazy }

??? Abstract "wikipedia"
    [X-ray crystallography](http://en.wikipedia.org/wiki/X-ray_crystallography) 
    
    [NMR](http://en.wikipedia.org/wiki/NMR) 
    
    [Mossbauer spectroscopy](http://en.wikipedia.org/wiki/Mossbauer_spectroscopy) 
    
    [Neutron scattering](http://en.wikipedia.org/wiki/Neutron_scattering) 
    
##### Simulation Methods to Study Protein Motions

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/simulation-methods-to-study-protein-motions-27493eb5'><i class='fa fa-play' aria-hidden='true'></i></button>Contrary to experimental techniques that only provide an average state of the sample, molecular simulations can deliver useful insights and precise details on the dynamics of each component of the protein ranging from the movement of single atoms to fragments, loops and domains. Molecular dynamics is the most commonly used simulation technique for studying a protein alone or interacting with other molecules. MD is a statistical mechanics method that follows the time course by integrating equations of motion. This method is described in some detail in the section "Molecular Dynamics".
*[Molecular dynamics]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system
*[MD]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system
*[statistical mechanics]: A method that treats the detailed state of a system as an ensemble subject to statistical uncertainties. It can be used to derive thermodynamic properties from properties at the molecular level.


![](https://media.drugdesign.org/course/structural-bioinformatics/ak_movie.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Molecular dynamics](http://en.wikipedia.org/wiki/Molecular_dynamics) 
    
    [Statistical mechanics](http://en.wikipedia.org/wiki/Statistical_mechanics) 
    
##### Normal Mode Analyses (NMA)

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/normal-mode-analyses-nma-eb8c5ae8'><i class='fa fa-play' aria-hidden='true'></i></button>"Normal mode analyses" is a method widely used in IR and Raman spectroscopy to analyze vibrational modes that can be used to study protein conformational changes. The method attracted renewed interest after the recent discovery that about 50% of protein movements can be modeled using a small number of low-frequency normal modes. Typically, it is used for the study of collective motions of complex biological systems.


![](https://media.drugdesign.org/course/structural-bioinformatics/nma3.gif){: loading=lazy }

??? PDB
    **PDB entry: 1hvr** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1hvr) 
    

??? Abstract "wikipedia"
    [Molecular vibration](http://en.wikipedia.org/wiki/Vibrational_spectroscopy) 
    
    [Normal mode](http://en.wikipedia.org/wiki/Normal_mode) 
    

???+ Abstract "Media"
    This picture was made using the WEBnm@ server (pdb:1hvr) 
    [Normal Mode Analysis](http://www.bioinfo.no/tools/normalmodes/)  
    
##### Molecular Dynamics vs Normal Mode Analyses
*[Molecular Dynamics]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/molecular-dynamics-vs-normal-mode-analyses-6a75c022'><i class='fa fa-play' aria-hidden='true'></i></button>In principle, Molecular Dynamics (MD) is better than Normal Mode Analyses (NMA) for the simulation of protein motions: the harmonic approximation introduced in NMA restricts the search to a small region of the conformational space. However, it has been observed that NMA produces reasonable results for a great number of systems. Moreover it has the advantage of being much less costly in terms of computing time than MD. For these two reasons NMA has become the preferred method for routine studies of protein motions aimed at describing protein dynamics in a concise way.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[Molecular Dynamics]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system
*[MD]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system
*[rms]: Root Mean Square: the square root of the arithmetic average of the square's set of values.
*[harmonic]: A function whose frequency is an integral multiple of the frequency of a reference function.


![](https://media.drugdesign.org/course/structural-bioinformatics/md_vs_nma.png){: loading=lazy }
#### Database of Macromolecular Movements

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/database-macromolecular-movements-bf853b57'><i class='fa fa-play' aria-hidden='true'></i></button>Marc Gerstein's group has compiled a database of macromolecular movements with associated tools for flexibility and geometric analysis. Conformational changes of more than 240 individual protein motions are described and visualized by movies. A web-based tool called the "Morph Server" enables the user to generate, interpolate and animate the transition between two distinct states of a macromolecular system (described as PDB files).
*[PDB]: The "Protein Data Bank". A worldwide repository source of 3D structures of proteins obtained by X-ray crystallography or NMR studies.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/db_gerstein.gif){: loading=lazy }


??? Abstract "wikipedia"
    [Database of Molecular Motions](
          ) 
    
## Assembly of Biomolecules

### Biological Molecule Association

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/biological-molecule-association-ecc98c15'><i class='fa fa-play' aria-hidden='true'></i></button>Every process in living cells evolves as the result of a complex association between molecules. Self assembly mainly occurs by specific molecular interactions. The ability to understand the nature and the driving forces behind these interactions is crucial in the life sciences.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/assembly_of_biomolecules.gif){: loading=lazy }


??? Abstract "articles"
    **A confederacy of bunches: Fundamentals and applications of a self-associating protein** 
    Wandless T. J. 
    Proc Natl Acad Sci USA 
    20(13) 2000 
         
    
### Molecular Recognition

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/molecular-recognition-84a0b3e5'><i class='fa fa-play' aria-hidden='true'></i></button>Biochemistry reveals to us how inanimate molecules that constitute living organisms recognize each other to maintain and perpetuate life. Understanding how molecules recognize each other is referred to as molecular recognition and is presented in the following pages.


![](https://media.drugdesign.org/course/structural-bioinformatics/molreg.gif){: loading=lazy }


??? Abstract "articles"
    **Molecular Docking: A Problem With Thousands Of Degrees Of Freedom** 
    M. Teodoro and G. Phillips and L. Kavraki 
    IEEE Press 
    
         2001 
         
    

??? Abstract "wikipedia"
    [Molecular Recognition](http://en.wikipedia.org/wiki/Molecular_recognition) 
    
#### The Recognition Process

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/recognition-process-9e998fea'><i class='fa fa-play' aria-hidden='true'></i></button>Molecular recognition is a multistep process. It begins by the approach of the molecules followed by desolvation of the interacting surfaces. The ligand starts to penetrate the active site of the protein. Orientational and conformational changes take place to form favorable interactions such as new hydrogen bonds, hydrophobic and electrostatic interactions. As a result, specific perturbations of the receptor induce the biological response.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/structural-bioinformatics/recognition_process.gif){: loading=lazy }

#### Complementary Features Upon Binding

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/complementary-features-upon-binding-8f3dfc8d'><i class='fa fa-play' aria-hidden='true'></i></button>One of the essential principles in molecular recognition is the structural complementarity between interacting molecules (known as the "lock-and-key" model). This was postulated in 1894 by Emil Fisher when he observed that invertase from beer yeast hydrolyzed &alpha;-glucosides but not &beta;-glucosides, while emulsin hydrolyzed the &beta;- but not the &alpha;-glucosides.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/complem_tole.gif){: loading=lazy }

#### Role of Native Protein Configuration

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/role-native-protein-configuration-89166213'><i class='fa fa-play' aria-hidden='true'></i></button>In 1936 Mirsky and Pauling demonstrated that the specific properties of native proteins were due to their uniquely defined configurations, whereas denaturated proteins lose these properties by lacking such organization. This idea was introduced twenty years before Anfisen's experiments on ribonuclease refolding and the X-ray determinations of the structure of myoglobin that finally led to the acceptance that highly stereospecific 3D structure is a fundamental requirement for protein function.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/mirsky.png){: loading=lazy }

??? Abstract "articles"
    **On the structure of native, denatured and coagulated proteins** 
    Mirsky AE, Pauling L 
    Proc. Natl. Acad. Sci. USA 
    22 1936 
           
    

??? Abstract "wikipedia"
    [Anfisen's Dogma](http://en.wikipedia.org/wiki/Anfinsen's_dogma) 
    
#### Tolerance Upon Binding
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/tolerance-upon-binding-70787020'><i class='fa fa-play' aria-hidden='true'></i></button>In 1950 Karush observed that many ligands of different shapes exhibited competitive binding to serum albumin. He extrapolated the lock and key principle and proposed that there is some tolerance upon binding, a process that favors the best-fitting member from an ensemble of structures in equilibrium. He called it "configurational adaptability".
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ligands]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[apo]: 3D structure of a macromolecule without a ligand


![](https://media.drugdesign.org/course/structural-bioinformatics/karush.png){: loading=lazy }

??? Abstract "articles"
    **Heterogeneity of the binding sites of bovine serum albumin** 
    Karush F 
    J. Am. Chem. Soc. 
    72 1950 
           
    
    **Intrinsically disordered protein** 
    Dunker A. K. et al. 
    Journal of Molecular Graphics and Modelling 
    19 2001 10.1016/S1093-3263(00)00138-8 
    
??? PDB
    **PDB entry: 1BM0** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BM0) 
    

???+ Abstract "Media"
    This picture was made using KiNG (Kinetic Image, Next Generation) 
    [KiNG Home page](http://kinemage.biochem.duke.edu)  
    
##### The "Induced-Fit" Theory

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/induced-fit-theory-ed56f47b'><i class='fa fa-play' aria-hidden='true'></i></button>Independently, Daniel Koshland introduced the "induced-fit" theory in 1958. This theory stipulates that in the recognition process, protein and ligand mutually adapt themselves by small conformational changes until an optimal fit is achieved. This model accounts for molecules that can bind despite having non-perfect complementarity in the unbound state.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


=== "Example"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/emovie.gif){: loading=lazy }

=== "Principle"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/induced_fit2.png){: loading=lazy }

??? Abstract "articles"
    **Remembering Daniel E. Koshland Jr. (1920-2007)** 
    Dahlquist F.W. 
    Protein Science 
    16 2007 
             
    
    **Complexes of alkylene-linked tacrine dimers with Torpedo californica acetylcholinesterase: Binding of bis(5)-tacrine produces a dramatic rearrangement in the active-site gorge** 
    Rydberg, E.H., Brumshtein, B., Greenblatt, H.M., Wong, D.M., Shaya, D., Williams, L.D., Carlier, P.R., Pang, Y.-P., Silman, I. and Sussman, J.L. 
    J. Med. Chem. 
    49 2006 
             
    
    **eMovie: A storyboard-based tool for making molecular movies** 
    Hodis, E., Schreiber, G., Rother, K. and Sussman, J.L. 
    Trends in Biochemical Sciences 
    32 2007 
             
    
??? PDB
    **PDB entry: 1HKG (apo)** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1hkg) 
    
    **PDB entry: 1GLK (complex)** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1GLK) 
    

??? Abstract "wikipedia"
    [The induced Fit Model](http://en.wikipedia.org/wiki/Lock-and-key_model_%28enzyme%29) 
    
##### Example of Enzyme Adaptation to Inhibitor Binding
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-enzyme-adaptation-to-inhibitor-binding-68c205a7'><i class='fa fa-play' aria-hidden='true'></i></button>An example of enzyme adaptation to inhibitor binding is described in the work of Gee et al (2007) on phenylethanolamine N-methyltransferase (PNMT), the enzyme that catalyzes the N-methylation of norepinephrine. They found that some inhibitors (such as SKF 29661) can substantially increase the size of the active site of this enzyme.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/transferase_example.png){: loading=lazy }

??? Abstract "articles"
    **Enzyme Adaptation to Inhibitor Binding: A Cryptic Binding Site in Phenylethanolamine N-Methyltransferase** 
    Gee C,L. et al. 
    J. Med. Chem. 
    50 2007 10.1021/jm0703385 
    
??? PDB
    **PDB entry: 1hnn** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1hnn) 
    
    **PDB entry: 2g72** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2g72) 
    

??? Abstract "wikipedia"
    [Phenylethanolamine N-methyltransferase](http://en.wikipedia.org/wiki/PNMT) 
    
##### Example of Ligand Adaptation upon Binding
*[Ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-ligand-adaptation-upon-binding-9ee82de8'><i class='fa fa-play' aria-hidden='true'></i></button>It is important to note that induced-fit adaptations not only apply to conformational changes of the receptor but also to those of the ligand. This is illustrated below in the binding of acetylcholine to the nicotinic acetylcholine receptor. The 3D structure of acetylcholine in its free solution-state conformation (left) is compared to the conformation of acetylcholine in its receptor-bound state (right).
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/acth.png){: loading=lazy }

??? Abstract "articles"
    **Conformation of acetylcholine bound to the nicotinic acetylcholine receptor** 
    RONALD W. BEHLING, TETSUO YAMANE, GIL NAVON, AND LYNN W. JELINSKI 
    Proc. Nati. Acad. Sci. USA 
    85 1988 
             
    

??? Abstract "wikipedia"
    [Nicotinic acetylcholine receptor](http://en.wikipedia.org/wiki/Nicotinic_acetylcholine_receptor) 
    
##### Maximizing Surface Contacts

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/maximizing-surface-contacts-d29140ad'><i class='fa fa-play' aria-hidden='true'></i></button>The aim of the induced-fit process is to maximize the common shared area of the interacting molecules. This phenomenon, usually referred to as the plasticity of the protein surface, contributes to the thermodynamic stability of the complexes. To better illustrate the 3D complementarity between the two proteins shown below, a "slab" option was used that shows cross sections of the system.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/maximize.gif){: loading=lazy }

##### Motions Associated to Induced-Fit

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/motions-associated-to-induced-fit-0ba2980a'><i class='fa fa-play' aria-hidden='true'></i></button>Motions associated to the induced-fit process include small backbone displacements and side-chain and loop rearrangements. Adaptation of the binding site to the shape of the ligand is mainly achieved through rearrangement of the side chains of the protein that can change their rotameric state with small energy changes. It has been observed that 40% of all side chains of a protein have a 10%-40% chance of undergoing a substantial rearrangement upon binding of the ligand.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/loop_3.png){: loading=lazy }

??? Abstract "articles"
    **Protein Flexibility and Drug Design: How to Hit a Moving Target** 
    Carlson, H.A. 
    Current Opinion in Chemical Biology 
    6 2002  
    
##### Experimental Evidence of the Induced-Fit Model
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/experimental-evidence-induced-fit-model-a62f1821'><i class='fa fa-play' aria-hidden='true'></i></button>In a work designed to provide experimental evidence for the induced-fit model Gutteridge and Thornton analyzed the X-ray structures of 60 different enzymes to see whether conformational changes could be observed between the apo and the complexed forms. They found that induced-fit motions in most enzymes were very small (less than 1 &#8491; RMSD across the whole protein). This confirms the validity of the induced-fit theory and demonstrates that small conformational changes have a major impact on the molecular recognition process.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[rms]: Root Mean Square: the square root of the arithmetic average of the square's set of values.
*[RMSD]: The Root Mean Square Deviation is a measure of the differences between values predicted and values experimentally observed.
*[apo]: 3D structure of a macromolecule without a ligand


![](https://media.drugdesign.org/course/structural-bioinformatics/induced_fit_experiment.png){: loading=lazy }

??? Abstract "articles"
    **Conformational changes observed in enzyme crystal structures upon substrate binding** 
    GUTTERIDGE A, THORNTON J: 
    J. Mol. Biol. 
    346(1) 2005 10.1016/j.jmb.2004.11.013 
    
    **Incorporating protein flexibility into docking and structure-based drug design** 
    Barril X, Fradera X 
    Expert Opin. Drug Discov. 
    1(4) 2006 10.1517/17460441.1.4.335 
    
#### Large Rearrangements

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/large-rearrangements-d828452b'><i class='fa fa-play' aria-hidden='true'></i></button>In addition to small induced-fit motions, larger rearrangements can be observed involving the relative motions of domains or substructural parts. This phenomenon was first observed in 1978 by Bennett and Steitz in the binding of glucose with hexokinase. The free enzyme is shown here in blue and the complex with glucose appears in red. Note the rearrangement that enabled the substrate to bind.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/hexakinase.gif){: loading=lazy }


??? Abstract "articles"
    **Glucose-induced conformational change in yeast hexokinase** 
    Bennett, W.S., Jr. and Steitz, T.A. 
    Proc. Natl. Acad. Sci. USA 
    75 1978 
           
    
    **Structural dynamics of yeast hexokinase during catalysis** 
    W.S.Bennett and T.A.Steitz 
    Philos Trans R Soc Lond B Biol Sci 
    293(1063) 1981 
           
    
    **A refined model of the sugar binding site of yeast hexokinase B.** 
    Anderson CM, Stenkamp RE, McDonald RC, Steitz TA. 
    J Mol Biol 
    123(2) 1978 
           
    
??? PDB
    **PDB entry: 2YHX** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2yhx) 
    
    **PDB entry: 1HKG** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1hkg) 
    

???+ Abstract "Media"
    This picture was made using the HingeProt server and VMD 
    [HingeProt Server](http://www.prc.boun.edu.tr/appserv/prc/HingeProt3/index.html)  
    
##### Role of Large Rearrangements

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/role-large-rearrangements-183c6145'><i class='fa fa-play' aria-hidden='true'></i></button>Large rearrangements are often key determinants in the mechanism and the function of a protein. For example the interconversion between the closed-inactive and the open-active conformations of an enzyme may enable the binding of the substrate or the release of a product.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/domain_motion.gif){: loading=lazy }

??? PDB
    **PDB entry: 4cln** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=4cln) 
    

???+ Abstract "Media"
    This picture was made using the HingeProt server and VMD 
    [HingeProt Server](http://www.prc.boun.edu.tr/appserv/prc/HingeProt3/index.html)  
    
#### The Domino Effect

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/domino-effect-a71abbdc'><i class='fa fa-play' aria-hidden='true'></i></button>In a multi-subunit protein a conformational change in one subunit can affect the conformation of the other subunits. The binding of a ligand to a protein may be regulated through specific interactions with one or more additional ligands. For example, these other ligands may cause conformational changes in the protein that affect the initial binding of the first ligand; this is the so-called "domino effect" sometimes observed in protein binding.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ligands]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/structural-bioinformatics/domino.png){: loading=lazy }
#### Proteins Described as Ensemble of Conformations

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/proteins-described-as-ensemble-conformations-95eb8206'><i class='fa fa-play' aria-hidden='true'></i></button>To mimic the plasticity inherent to all proteins which enables them to interact with and adapt to other proteins, a protein can be described as an ensemble of very different conformations. Moreover, a ligand can recognize and bind to any conformation and subsequently, the induced-fit mechanism occurs to maximize the corresponding binding.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/structural-bioinformatics/multiple_state3.png){: loading=lazy }

??? Abstract "articles"
    **FOLDING FUNNELS AND BINDING MECHANISMS** 
    BUYONG MA ; KUMAR S. ; TSAI C.-J. ; NUSSINOV R. 
    Protein eng. 
    12(9) 1999  
    
#### Energy Landscape of a Protein

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/energy-landscape-protein-d1396a9a'><i class='fa fa-play' aria-hidden='true'></i></button>The profile of the potential energy surface of a protein is therefore represented by a great number of funnels, each one corresponding to a family of conformations, of which only a small number play a biological role and the remainder merely reflect the plasticity of the protein.


![](https://media.drugdesign.org/course/structural-bioinformatics/protein_path.png){: loading=lazy }

??? Abstract "articles"
    **FOLDING FUNNELS AND BINDING MECHANISMS** 
    BUYONG MA ; KUMAR S. ; TSAI C.-J. ; NUSSINOV R. 
    Protein eng. 
    12(9) 1999  
    
    **IMPLICATIONS OF PROTEIN FLEXIBILITY FOR DRUG DISCOVERY** 
    Simon J. Teague 
    Nat Rev Drug Discov 
    2 2003  
    
#### Conformational Selection Operated by a Ligand
*[Ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/conformational-selection-operated-by-ligand-f6955eb9'><i class='fa fa-play' aria-hidden='true'></i></button>The recognition process, in which the ligand binds selectively to a given conformation of the protein, is called "conformational selection". The conformational selection operated by the ligand increases the population of the selected conformation of the protein with respect to the other conformations and leads to the so-called "energetic induction" phenomenon.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/structural-bioinformatics/ligand_design.png){: loading=lazy }

??? Abstract "articles"
    **IMPLICATIONS OF PROTEIN FLEXIBILITY FOR DRUG DISCOVERY** 
    Simon J. Teague 
    Nat Rev Drug Discov 
    2 2003  
    
#### Energetic Induction Upon Binding

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/energetic-induction-upon-binding-bd067c62'><i class='fa fa-play' aria-hidden='true'></i></button>The binding of a ligand to the protein results in an energetic induction phenomenon where the potential surface of the protein and the preference between the conformational minima are altered.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/protein_induction.gif){: loading=lazy }

### Forces Involved in Molecular Recognition

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/forces-involved-molecular-recognition-a6e59b06'><i class='fa fa-play' aria-hidden='true'></i></button>Molecular interactions are responsible for the assembly of biological structures. A correct understanding of the forces that govern such interactions is important for gaining insight into molecular associations. These forces include Van der Waals interactions, electrostatic interactions and the formation of hydrogen bonds.


![](https://media.drugdesign.org/course/structural-bioinformatics/chemical_interaction.png){: loading=lazy }

??? Abstract "wikipedia"
    [Intermolecular force](http://en.wikipedia.org/wiki/Intermolecular_force) 
    
#### Van der Waals Forces

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/van-der-waals-forces-347f2fa7'><i class='fa fa-play' aria-hidden='true'></i></button>Van der Waals (VdW) forces between atoms are due to temporary atomic induced polarization between interacting atoms ("induced dipoles"). At long distances, individually, these forces are very weak, but the accumulation of a large number of such weak forces substantially contributes to a molecular association. At short distances, the van der Waals forces become strongly repulsive because of the steric clash occurring with the overlap of the electron clouds. The Lennard Jones (6-12) potential describes these two types of VdW interactions.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/vanderwaals.png){: loading=lazy }
=== "d=6"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/vanderwaals_f1.png){: loading=lazy }
=== "d=4"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/vanderwaals_f2.png){: loading=lazy }
=== "d=3"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/vanderwaals_f3.png){: loading=lazy }
=== "d=2"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/vanderwaals_f4.png){: loading=lazy }
=== "d=1.5"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/vanderwaals_f5.png){: loading=lazy }

??? Abstract "wikipedia"
    [Lennard-Jones potential](http://en.wikipedia.org/wiki/Lennard-Jones_potential) 
    
    [Van der Waals force](http://en.wikipedia.org/wiki/Van_der_Waals_force) 
    
#### Electrostatic Interactions

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/electrostatic-interactions-da7ced79'><i class='fa fa-play' aria-hidden='true'></i></button>Electrostatic interactions are strong forces acting between charged particles. They can be attractive (if the two charges are unlike) or repulsive (if the charges are like). The electrostatic potential can be simply described by Coulomb's law. It decays slowly as a function of the distance between the charges and therefore can still be meaningful at relatively long distances. In order to accurately describe the electrostatic potential it is preferable to locate partial charges and not only the net charge of the atoms.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/electrostatic.png){: loading=lazy }


??? Abstract "wikipedia"
    [Coulomb's law](http://en.wikipedia.org/wiki/Coulomb_law) 
    
#### Hydrogen Bonds

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/hydrogen-bonds-fc549ec7'><i class='fa fa-play' aria-hidden='true'></i></button>The hydrogen bond is perhaps the most significant non-bonded interaction in biology. It accounts for the double helix organization of two complementary DNA strands and also plays an important role in all types of macromolecules. For example, C=O---NH hydrogen bonds of the backbone of a protein account for the organization of its secondary and tertiary structure. Typically the interaction occurs between one hydrogen bond donor (e.g. NH, OH) and one acceptor (e,g. O, N) that share the hydrogen between them in a well- defined geometry.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/alpha_helix.png){: loading=lazy }


??? Abstract "wikipedia"
    [Hydrogen bond](http://en.wikipedia.org/wiki/Hydrogen_bonds) 
    
#### Solvent Effect

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/solvent-effect-67b7c755'><i class='fa fa-play' aria-hidden='true'></i></button>The majority of molecular interactions occur in the liquid phase; therefore one must consider the solvent (often water) and its interactions with the surrounding molecules. In particular energies need to be expended for moving the polar part of a ligand out of the water and into the binding site.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/liquid_phase.png){: loading=lazy }
##### The Role of the Solvent

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/role-solvent-7582ab37'><i class='fa fa-play' aria-hidden='true'></i></button>Solvent molecules play many roles. First, there are direct interactions between solvent and solute molecules. Moreover, apolar molecules tend to aggregate in the presence of water. This effect is known to be entropically rather than enthalpically driven and is known to play a major role in dictating the folding pattern of proteins. Solvent molecules can also screen electrostatic interactions in the medium or form intermolecular bridges between a ligand and a protein.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[apo]: 3D structure of a macromolecule without a ligand


![](https://media.drugdesign.org/course/structural-bioinformatics/solvent_effects.png){: loading=lazy }
##### The Hydrophobic Effect

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/hydrophobic-effect-9edfa33e'><i class='fa fa-play' aria-hidden='true'></i></button>The aqueous solvent plays a crucial role in determining the conformational preferences and interaction forces of ligands and receptors. In particular, the hydrophobic effect (also called hydrophobic collapse) is caused by the unfavorable interaction of non-polar groups with water molecules; they tend to cluster together to exclude water and minimize the area of interaction with water. The interpretation of this phenomenon is that water-water interactions are the most favorable interactions and in effect, the water squeezes out the hydrophobes so it can interact with itself, leaving the hydrophobes to interact with themselves.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ligands]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/hydro_effect.png){: loading=lazy }

??? PDB
    **PDB entry: 2TRX** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2TRX) 
    
##### The Entropic Effects

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/entropic-effects-5ccdfad7'><i class='fa fa-play' aria-hidden='true'></i></button>The free energy of a system is composed of two components: enthalpy (or potential energy) and entropy (or "degree of disorder"). The solvent controls the entropy of a biological system by influencing the conformations of the interacting molecules. A flexible molecule has a better chance of finding an optimal fit into a receptor, but this is achieved at the cost of large conformational entropy.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/entropic_effects2.png){: loading=lazy }

??? Abstract "articles"
    **Ligand configurational entropy and protein binding** 
    Chang C.A et al. 
    PNAS 
    104 2007 
             
    
##### Enthalpy-Entropy Compensation

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/enthalpy-entropy-compensation-9b01610f'><i class='fa fa-play' aria-hidden='true'></i></button>Many aspects of complex formation are unfavorable as they result in the loss of conformational degrees of freedom for the interacting molecules; therefore, highly favorable enthalpic contacts must compensate for this entropic loss.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/entropic_effects.png){: loading=lazy }

??? Abstract "wikipedia"
    [Enthalpy-entropy compensation](http://en.wikipedia.org/wiki/Enthalpy-entropy_compensation) 
    
### Assessing Binding Interactions

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/assessing-binding-interactions-e0728cc3'><i class='fa fa-play' aria-hidden='true'></i></button>Now that the forces that dictate molecular associations are well characterized, we can quantitatively assess them. This will pave the way to molecular simulations and the prediction of binding interactions and binding energies.


![](https://media.drugdesign.org/course/structural-bioinformatics/binding_energy.png){: loading=lazy }
#### Free Energy of Binding
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/free-energy-binding-e57fa9dd'><i class='fa fa-play' aria-hidden='true'></i></button>Binding energy is the energy required to disassemble a whole system into separate parts; it represents the mechanical work which must be done by acting against the forces which hold the molecules together. A complex has a lower energy than its constituent parts. Estimating the binding energy is a central issue in biology and structural bioinformatics.


![](https://media.drugdesign.org/course/structural-bioinformatics/free_energy4.png){: loading=lazy }

??? Abstract "wikipedia"
    [Binding energy](http://en.wikipedia.org/wiki/Binding_energy) 
    
#### Importance of Free Energy of Binding
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/importance-free-energy-binding-e58bbd19'><i class='fa fa-play' aria-hidden='true'></i></button>From a chemist's point of view, free energies (also called Gibbs energies &Delta;G) are of utmost importance because differences in free energies determine binding constants, equilibrium rates, conformations and many other properties of a system. &Delta;G can be obtained experimentally or theoretically. Experimentally, this requires the availability of the two interacting molecules; theoretically, this is not needed at all, however they often need to approximate &Delta;G by the potential energy of the system.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/free_energy.png){: loading=lazy }

??? Abstract "wikipedia"
    [Thermodynamic free energy](http://en.wikipedia.org/wiki/Thermodynamic_free_energy) 
    
    [Gibbs free energy](http://en.wikipedia.org/wiki/Gibbs_free_energy) 
    
#### Experimental Measures of Binding Affinities

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/experimental-measures-binding-affinities-2e72d291'><i class='fa fa-play' aria-hidden='true'></i></button>The association of two biomolecules is an equilibrium process. The binding to a receptor is a phenomenon where the two molecules associate and dissociate, a process regulated by the law of mass action. A measure of the "goodness of fit" of a complex is measured by the Kd constant (dissociation constant) of the equilibrium. The dissociation constant has molar units (M); a nanomolar complex (Kd ~ nM) is more tightly bound than a micromolar one (Kd ~ &micro;M).


![](https://media.drugdesign.org/course/structural-bioinformatics/k_mesear.png){: loading=lazy }

??? Abstract "wikipedia"
    [Law of mass action](http://en.wikipedia.org/wiki/Law_of_mass_action) 
    
##### Titration Curve to Measure Kd

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/titration-curve-to-measure-kd-fce1e197'><i class='fa fa-play' aria-hidden='true'></i></button>The dissociation constant corresponds to the concentration of a ligand required to occupy 50% of its binding site on a particular protein. Experimentally, the dissociation constant of a system is measured by plotting the titration curve giving the percentage of occupation of the protein &#920; as a function of the concentration of the ligand [L]. Kd is obtained by finding the point for which &#920; = 50% in the graph.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.


![](https://media.drugdesign.org/course/structural-bioinformatics/calculation_of_kd.png){: loading=lazy }

??? Abstract "wikipedia"
    [Dissociation constant](http://en.wikipedia.org/wiki/Dissociation_constant) 
    
##### Scatchard-Rosenthal Plots

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/scatchard-rosenthal-plots-c684164a'><i class='fa fa-play' aria-hidden='true'></i></button>A convenient way to obtain Kd consists of linearizing the titration curve obtained on the previous page and plotting &#920;/[L] as a function of &#920;. This is the so-called Scatchard-Rosenthal plot in which Kd is the negative reciprocal of the slope. With such plots, it is easy to compare two sets of data; if the slopes for both sets are similar, the Kd will be similar.


![](https://media.drugdesign.org/course/structural-bioinformatics/k_mesear3.png){: loading=lazy }

??? Abstract "wikipedia"
    [Scatchard equation](http://en.wikipedia.org/wiki/Scatchard_equation) 
    
##### Conversion of Kd into Energies

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/conversion-kd-into-energies-4efded66'><i class='fa fa-play' aria-hidden='true'></i></button>The relationship between the free energy and Kd is given by the equation &Delta;G = -RT LogKd. In this equation, R is the ideal gas constant, T is the temperature in Kelvin and Kd is the dissociation constant. Kd constants can be converted into energies, and vice versa. The following diagram represents &Delta;G (in kJ/mol) as a function of log Kd at room temperature.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/k_mesear4.png){: loading=lazy }
#### Theoretical Prediction of Binding Energies

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/theoretical-prediction-binding-energies-a9bad2a5'><i class='fa fa-play' aria-hidden='true'></i></button>The theoretical prediction of the binding energy of a system is of central importance for understanding biological mechanisms and this is also exploited in drug discovery. For example it enables a better understanding of why one ligand is preferred to another, and provides efficient means to speed the discovery of new molecules that bind tightly and specifically to particular proteins.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/theoretical.png){: loading=lazy }

??? Abstract "articles"
    **Design and Synthesis of a Series of Non-Peptide High-Affinity Human Corticotropin-Releasing Factor1 Receptor Antagonists** 
    Chen Chen, Raymond Dagnino, Jr., Errol B. De Souza, Dimitri E. Grigoriadis, Charles Q. Huang, Kyung-Il Kim, Zhengyu Liu, Terry Moran, Thomas R. Webb, Jeffrey P. Whitten, Yun Feng Xie, and James R. McCarthy 
    J. Med. Chem. 
    39 1996 10.1021/jm960149e 
    
##### Solving the Schrodinger Equation

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/solving-schrodinger-equation-7de66a4e'><i class='fa fa-play' aria-hidden='true'></i></button>From a theoretical point of view quantum mechanics and its associated Schrodinger equation gives the exact description of a system and enables the calculation of energies, energy levels, wave functions and many other properties. It can be solved exactly for systems of 2 or 3 atoms, whereas for systems of fewer than 100 atoms, although it can be still used, approximations need to be introduced. The calculation of macromolecular binding energies with quantum mechanics is far beyond the current state of the art in this field.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/schrodinger.png){: loading=lazy }

??? Abstract "wikipedia"
    [Schrodinger equation](http://en.wikipedia.org/wiki/Schr%C3%B6dinger_equation) 
    

???+ Abstract "Media"
    The Schrodinger picture is from the Wikimedia Commons (Public Domain) 
    [Erwin Schrodinger](http://commons.wikimedia.org/wiki/Image:Erwin_Schr%C3%B6dinger.jpg)  
    
    Adapted from K. Wilson UCSD 
    [The Schrodinger Equation](http://www.chemistry.ucsc.edu/teaching/switkes/CHEM163A/Fall02/KWILSON/qm/61.SchrodingerEquation.jpeg)  
    
##### Molecular Mechanics
*[Molecular Mechanics]: Molecular mechanics is a is a computer simulation technique for modeling the molecular geometry and energy of a system based on the energy minimization of its potential energy function. The set of potential functions used to calculate the energy is known as the force-field.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/molecular-mechanics-8c7b6c3d'><i class='fa fa-play' aria-hidden='true'></i></button>Molecular mechanics remains an alternative for calculating the binding energies of large molecular systems. The method considers atoms as spheres, centered at the position of their nucleus and applies the laws of classical mechanics to calculate the potential energy of the system. Molecular mechanics gives no information about absolute free energies; however, it is useful for comparing the relative energy of two different configurations.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[Molecular mechanics]: Molecular mechanics is a is a computer simulation technique for modeling the molecular geometry and energy of a system based on the energy minimization of its potential energy function. The set of potential functions used to calculate the energy is known as the force-field.


![](https://media.drugdesign.org/course/structural-bioinformatics/djhksd.png){: loading=lazy }

??? Abstract "wikipedia"
    [Molecular mechanics](http://en.wikipedia.org/wiki/Molecular_mechanics) 
    
##### Force-Field
*[Force-Field]: A set of mathematical equations and parameters used for assessing the energy of a given chemical system.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/force-field-d64ce1c5'><i class='fa fa-play' aria-hidden='true'></i></button>A "force-field" consists of a set of mathematical equations that simulate intra and inter-molecular forces which enable the calculation of the potential energy of a system. Forces and energies are directly related: the force is the negative derivative of the potential energy. Each component of the force-field models a particular force applied to the system. The sum of all these forces include stretching, bending, torsional, van der Waals, hydrogen bonds and electrostatic energies, and their total is an approximation of the total free energy &Delta;G that governs the system.
*[force-field]: A set of mathematical equations and parameters used for assessing the energy of a given chemical system.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/mm.png){: loading=lazy }

??? Abstract "wikipedia"
    [Force field](http://en.wikipedia.org/wiki/Force_field_%28chemistry%29) 
    
##### Example of Force-Fields
*[Force-Field]: A set of mathematical equations and parameters used for assessing the energy of a given chemical system.
*[Force-Fields]: A set of mathematical equations and parameters used for assessing the energy of a given chemical system.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/example-force-fields-89ce503e'><i class='fa fa-play' aria-hidden='true'></i></button>Force-fields can be used to measure binding energies. The most commonly used force-fields for macromolecules are Amber, Gromos, and Charmm. Special scoring functions were developed to experimentally reproduce known binding energies. This is discussed in some detail in different parts of this course, in particular in the chapters on molecular docking and molecular dynamics (Free Energy Perturbation method).
*[Force-field]: A set of mathematical equations and parameters used for assessing the energy of a given chemical system.
*[Force-fields]: A set of mathematical equations and parameters used for assessing the energy of a given chemical system.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[molecular dynamics]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system


![](https://media.drugdesign.org/course/structural-bioinformatics/charmm.png){: loading=lazy }

??? Abstract "wikipedia"
    [AMBER](http://en.wikipedia.org/wiki/AMBER) 
    
    [CHARMM](http://en.wikipedia.org/wiki/CHARMM) 
    
    [GROMOS](http://en.wikipedia.org/wiki/GROMOS) 
    
##### Other Methods

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/other-methods-7ed28726'><i class='fa fa-play' aria-hidden='true'></i></button>Other methods that can be used to estimate protein binding affinities also exist. The most accurate approaches are usually more computationally costly. Linear Interaction Energies and Free Energy Perturbation are the most rigorous methods for calculating binding energies. They take advantage of the properties of thermodynamic cycles and aim to predict the free energies of the binding of new ligands from knowledge of the experimental binding energy of a known reference compound.
*[nce]: New Chemical Entity: a compound not previously described.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ligands]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/scoring_methods.png){: loading=lazy }

??? Abstract "wikipedia"
    [Free energy perturbation](http://en.wikipedia.org/wiki/Free_energy_perturbation) 
    
    [Thermodynamic cycle](http://en.wikipedia.org/wiki/Thermodynamic_cycles) 
    
##### Incorporation of the Solvent

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/incorporation-solvent-4033418f'><i class='fa fa-play' aria-hidden='true'></i></button>Solvent effects remain a central problem in modeling calculations. Simplified models have been identified to be a major cause of inaccuracy in binding energy calculations. The most rigorous methods consider explicit solvent molecules but they are too slow. Other approaches represent the solvent as a continuum. These issues are discussed in detail in the chapter on molecular dynamics.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[molecular dynamics]: Molecular dynamics (MD) is a computer simulation technique which follows the time evolution of a molecular system in 3D. Successive integration of Newtons equations of motion over time enables to obtain information about time-dependent properties of the system


![](https://media.drugdesign.org/course/structural-bioinformatics/solvation.png){: loading=lazy }
## Obtaining Macromolecular 3D-Structures

### Experimental Methods

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/experimental-methods-7ad07bbb'><i class='fa fa-play' aria-hidden='true'></i></button>Knowing the exact 3D-structure of bio-molecules is essential for any attempt to understand how these machineries fulfill their function and how they interact with each other. Unfortunately, getting to the structure of such macromolecules in the lab is not easy. X-ray crystallography, Nuclear Magnetic Resonance (NMR) spectroscopy, and cryo-electron microscopy are the three main techniques that provide protein structures experimentally. The distribution by source of the structures available in the protein data bank is indicated in the diagram below.
*[Nuclear Magnetic Resonance]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[protein data bank]: The "Protein Data Bank". A worldwide repository source of 3D structures of proteins obtained by X-ray crystallography or NMR studies.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/pdb_stat2_d.png){: loading=lazy }
#### X-ray Crystallography

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/x-ray-crystallography-9d3f59f7'><i class='fa fa-play' aria-hidden='true'></i></button>X-ray crystallography is a technique in which the pattern generated by the diffraction of X-rays passing through the single crystal lattice of a given material is recorded and then analyzed to reveal the 3D structure of the molecules composing the lattice. X-ray crystallography involves several steps that are presented in the following pages.


![](https://media.drugdesign.org/course/structural-bioinformatics/x_ray.png){: loading=lazy }

??? Abstract "wikipedia"
    [X-ray crystallography](http://en.wikipedia.org/wiki/X-ray_crystallography) 
    
##### Protein Production and Purification

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/protein-production-purification-8ac1d2a6'><i class='fa fa-play' aria-hidden='true'></i></button>The first step is to get large amounts of protein we want to crystallize. This is generally done by cloning the protein in various expression systems such as microorganism, bacteria, plant etc.... The protein is then purified, since only highly purified solutions can give rise to crystals and not every protein can be purified to a sufficient degree.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/purify.png){: loading=lazy }
##### Growing of Single Crystal

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/growing-single-crystal-1d515c0c'><i class='fa fa-play' aria-hidden='true'></i></button>The next step is the growth of single crystals. Many approaches are used to produce crystals but the general method is to start with a solution of the protein or its complex and to slowly change the conditions so that saturation is exceeded. This can be done by changing parameters such as ionic strength, pH, temperature of the solution and the addition of seed molecules. Supersaturation has to be reached slowly so that only a few nucleation sites are created which, with time, will grow larger. Thousands of solution conditions may be tried before finding one that succeeds in crystallizing the molecules, and ultimate success is never guaranteed.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/growing_crystal.png){: loading=lazy }
##### The Single Crystal

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/single-crystal-1820c931'><i class='fa fa-play' aria-hidden='true'></i></button>The starting point of X-rays is a single crystal (or a monocrystal) that must be a crystalline solid in which the crystal lattice of the sample is pure, continuous and unbroken. The basic unit of a crystal is called the unit cell, which is defined by three principal axes (a, b and c) and three angles (alpha, beta and gamma). Single crystals are difficult to obtain in the laboratory. A protein crystal needs to be at least 0.2-0.3 mm in all three dimensions to diffract strongly enough to achieve the resolution needed for a successful 3D structure determination.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/xray_cell.png){: loading=lazy }
##### Collecting the Diffraction Data

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/collecting-diffraction-data-d5525a4b'><i class='fa fa-play' aria-hidden='true'></i></button>The crystal is then bombarded by X-rays, and the interference between rays scattered by different parts of the crystal give rise to diffraction. This diffraction pattern is dependent on the 3D location of the atoms. Diffraction data only record intensity; unfortunately, all information about the phase angle (which relates the various diffraction points and the unit cell) is lost. This is a major hurdle for structure determination.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Principle"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/pattern.png){: loading=lazy }
=== "Diffractometer"
    ![](https://media.drugdesign.org/course/structural-bioinformatics/x_ray_machine.png){: loading=lazy }
##### Recovering the Phase Angle

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/recovering-phase-angle-69981764'><i class='fa fa-play' aria-hidden='true'></i></button>In order to solve the structure the lost phase angle must be recovered. Many strategies have been developed and include multiwavelength anomalous diffraction (MAD), multiple isomorphous replacement (MIR) and molecular replacement methods.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/derive_phase.png){: loading=lazy }

??? Abstract "wikipedia"
    [Phase problem](http://en.wikipedia.org/wiki/Phase_problem) 
    
    [Multi-wavelength anomalous dispersion](http://en.wikipedia.org/wiki/Multi-wavelength_anomalous_dispersion) 
    

???+ Abstract "Media"
    
             
    [Synchrotron - Copyright Chaix and Morel et associes, used with permission](http://en.wikipedia.org/wiki/Image:Sch%C3%A9ma_de_principe_du_synchrotron.jpg)  
    
##### Structure Determination and Refinement

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/structure-determination-refinement-f2fb00c8'><i class='fa fa-play' aria-hidden='true'></i></button>On the basis of the measured intensities of the diffraction points and the recovered phase angles, electron density maps are calculated. The maps are then interpreted and fitted to a 3D structure model. The quality of the structure can be evaluated by its resolution and by comparing the observed diffraction pattern and the one calculated from the final model. On the basis of this comparison the structure can be iteratively refined.


![](https://media.drugdesign.org/course/structural-bioinformatics/refinement.png){: loading=lazy }
##### Atomic Coordinates

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/atomic-coordinates-7c2279cc'><i class='fa fa-play' aria-hidden='true'></i></button>At the end of the process, the 3D structure of the macromolecule is obtained as a list of coordinates (x, y and z). The overall quality of the coordinates is usually measured by two quantities: (1) the resolution; this is the ability to resolve two points separated by some distance from each other. The higher the resolution, the "sharper" our view of the structure. High resolutions of macromolecules are around 2 &#8491;; (2) the crystallographic R factor; it measures the agreement between observed structure factors and those calculated from the model. A well-refined structure of a macromolecule has an R factor of not larger than 15%.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/pdb_coordinate.png){: loading=lazy }
##### The Advantages of X-ray Crystallography

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/advantages-x-ray-crystallography-e3bbc285'><i class='fa fa-play' aria-hidden='true'></i></button>X-ray crystallography has major advantages, making it the most practical method for bio-molecule structure determination. More importantly, it provides accurate atomic resolution models and allows scientists to resolve relatively large structures and complexes. Recent examples are structures of viral capsides and the ribosome, each composed of tens of thousands of atoms.


![](https://media.drugdesign.org/course/structural-bioinformatics/capside.png){: loading=lazy }

??? PDB
    **PDB entry: 1qgt** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1qgt) 
    

???+ Abstract "Media"
    
             
    [This picture was made using the PDB SimpleViewer 1.5](http://www.pdb.org/)  
    
##### The Limitations of X-ray Crystallography

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/limitations-x-ray-crystallography-9c291d91'><i class='fa fa-play' aria-hidden='true'></i></button>X-ray crystallography also has some major limitations: (1) the information provides only one snapshot of the protein that does not reflect its dynamic behavior; (2) contacts between molecules in the crystal and the dense packing might affect the structures; (3) the procedure is very slow; (4) there are many macromolecules which can not be resolved, especially proteins with significant hydrophobic portions (membrane proteins).
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/xray_disavantage.png){: loading=lazy }
#### NMR Spectroscopy
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/nmr-spectroscopy-98675499'><i class='fa fa-play' aria-hidden='true'></i></button>Nuclear magnetic resonance (NMR) is a physical phenomenon based upon the magnetic properties of the nuclei of some atoms. NMR spectroscopy was developed in the mid-1980s enabling the elucidation of 3D structures of macromolecules in solution. The first NMR structures deposited at the PDB started to appear in the early 1990s.
*[Nuclear magnetic resonance]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[PDB]: The "Protein Data Bank". A worldwide repository source of 3D structures of proteins obtained by X-ray crystallography or NMR studies.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/nmr_principle.png){: loading=lazy }

???+ Abstract "Media"
    This picture was released into the public domain 
    [wikipedia](http://en.wikipedia.org/wiki/Image:HWB-NMRv900.jpg)  
    
##### NMR Concepts
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/nmr-concepts-83d99edc'><i class='fa fa-play' aria-hidden='true'></i></button>NMR spectroscopy takes advantage of the fact that some nuclei have spins. The spinning charges generate a magnetic field that will either align with or against an applied external magnetic field. Absorption of energy supplied by electromagnetic radiation at specific frequencies (resonance) can cause the spin to reverse its orientation. The relaxation process of the spins can help to characterize the system. The most basic contribution to the resonance of a particular nucleus is the local electronic environment. For example electronegative atoms or delocalized electrons in aromatic rings will cause a shift of the resonance frequency in neighboring spin systems ("chemical shifts").
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/nmr_spin.png){: loading=lazy }
##### Spin-Spin Coupling
*[Spin-Spin Coupling]: The interaction between the spin magnetic moments of different particles (electrons, nuclei) that gives a detectable signal in NMR (multiplet patterns).

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/spin-spin-coupling-503478a9'><i class='fa fa-play' aria-hidden='true'></i></button>The interactions between neighboring spins ("spin-spin coupling") give rise to the line splitting in NMR spectra. These interactions provide connectivity information because they occur only between protons on covalently bonded atoms. Another spin-spin interaction can occur between non-covalently bonded nuclei in space (the Nuclear Overhauser Effect - "NOE"). This interaction is distance dependent and therefore only protons separated by less than 5&#8491; can be observed. This is the most crucial information for structure determination.
*[spin-spin coupling]: The interaction between the spin magnetic moments of different particles (electrons, nuclei) that gives a detectable signal in NMR (multiplet patterns).
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/spin_spin_coupling.png){: loading=lazy }
##### Data Collection

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/data-collection-0161ed94'><i class='fa fa-play' aria-hidden='true'></i></button>Since the signal from the NMR experiment is exceedingly weak, repeated scans are necessary to improve the signal-to-noise ratio. Signals arising from the decay of all resonances are measured simultaneously. This complex signal is then deconvoluted by the use of Fourier transforms into a set of frequencies and their corresponding amplitudes. Classical 1D NMR is useless for analyzing the 3D structure of macromolecules (due to the spectral overlap of hundreds of protons) and therefore 2D and 3D NMR techniques are used. Correlated spectroscopy (COSY) gives rise to cross peaks corresponding to the covalent structure (spin-spin couplings) while in NOESY cross peaks are obtained from the NOE effect.
*[spin-spin coupling]: The interaction between the spin magnetic moments of different particles (electrons, nuclei) that gives a detectable signal in NMR (multiplet patterns).
*[spin-spin couplings]: The interaction between the spin magnetic moments of different particles (electrons, nuclei) that gives a detectable signal in NMR (multiplet patterns).
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[sar]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.
*[rms]: Root Mean Square: the square root of the arithmetic average of the square's set of values.


![](https://media.drugdesign.org/course/structural-bioinformatics/nmr_data_collection.png){: loading=lazy }
##### Structure Determination

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/structure-determination-a240fe1e'><i class='fa fa-play' aria-hidden='true'></i></button>Once all the resonances have been assigned, the outcome is a list of rough inter-proton distances from the intensity of the NOE signal between interacting protons and some dihedral angles from spin-spin coupling constants. 3D models consistent with the set of constraints are then generated. These models (usually several dozen) can be refined by calculating NOEs from the model and fitting against the measured NOE intensities, as well as by optimizing for more favorable structural properties.
*[spin-spin coupling]: The interaction between the spin magnetic moments of different particles (electrons, nuclei) that gives a detectable signal in NMR (multiplet patterns).
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/nmr_stucture_determ.png){: loading=lazy }
##### Analysis

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/analysis-100c12a7'><i class='fa fa-play' aria-hidden='true'></i></button>Finally, when an ensemble of refined structures is obtained which satisfies the experimental restraints and has reasonable stereo-chemical parameters, we are ready to analyze the data. From the ensemble one can, for example, calculate deviations in the position of individual atoms, derived from the multiples structures. This information is believed to be related to the dynamics of the structure in solution. The quality of the final structure depends on both the number and quality of the NOE measurements.


![](https://media.drugdesign.org/course/structural-bioinformatics/nmr_analysis.png){: loading=lazy }
##### The Advantages of NMR
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/advantages-nmr-2bfefcf4'><i class='fa fa-play' aria-hidden='true'></i></button>The main advantage of NMR spectroscopy over X-ray crystallography is that there is no need to have the sample in a crystalline state, which restricts the applicability of X-ray crystallography. Crystal contacts may also affect the structure. The positions of all atoms, including hydrogens, are obtained. Crystal structures positions of hydrogens can only be determined at very high resolution. Finally, information about the dynamics of the molecule in solution can be deduced.
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/nmr_animation2.gif){: loading=lazy }

??? PDB
    **PDB entry: 1a5j** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1a5j) 
    
##### The Limitations of NMR
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/limitations-nmr-d2eb528c'><i class='fa fa-play' aria-hidden='true'></i></button>NMR based approaches for structure determinations have some major limitations. The precision of the technique is still not as high as in X-ray crystallography. In addition, only relatively small bio-molecules (for proteins usually smaller than 300 residues) can be solved. According to the size of the putative proteins (Open Reading Frames - ORFs) encoded in different genomes (see bars in different colors in the figure), less than half of the proteins can be solved by current NMR techniques (the limit is indicated by the dashed red line in the figure). Rapid developments in equipment and software has led however, to continual improvement in these areas.
*[NMR]: An analytical method that allows the spectroscopic detection of structural information of molecules. It requires the application of a strong magnetic field.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/nmr_limitations.png){: loading=lazy }

??? Abstract "articles"
    **NMR spectroscopy: pushing the limits of sensitivity** 
    Spiess HW 
    Angew Chem Int Ed Engl 
    47 (4) 2008 
             
    
    **The structure of the neuropeptide bradykinin bound to the human G-protein coupled receptor bradykinin B2 as determined by solid-state NMR spectroscopy** 
    Lopez JJ, Shukla AK, Reinhart C, Schwalbe H, Michel H, Glaubitz C 
    Angew Chem Int Ed Engl 
    47 (9) 2008 
             
    
#### Electron Microscopy

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/electron-microscopy-ceff6b03'><i class='fa fa-play' aria-hidden='true'></i></button>Electron microscopy (EM) is another approach that provides information about the 3D structure of macromolecules. It has recently become popular in structural biology as it can provide molecular structural data about large molecular complexes.
*[structural biology]: Structural Biology is a new field of modern biology dedicated to the determination of the 3D structure of biological proteins to give insight to the function of these macromolecules in the living cell and to exploit this knowledge for the design of new drugs.


![](https://media.drugdesign.org/course/structural-bioinformatics/electron_micro.png){: loading=lazy }

??? PDB
    **PDB entry: 3byh** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3byh) 
    

??? Abstract "wikipedia"
    [Electron microscopy](http://en.wikipedia.org/wiki/Electron_microscope) 
    

???+ Abstract "Media"
    This picture was taken from the RCSB PDB website 
    [PDB Protein Databank](http://www.rcsb.org/pdb/explore.do?structureId=3BYH)  
    
##### Basic Concept
*[nce]: New Chemical Entity: a compound not previously described.

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/basic-concept-7a42ea8d'><i class='fa fa-play' aria-hidden='true'></i></button>The electron microscope uses electrons to create an image of the target. It has much higher magnification or resolving power than a regular light microscope, allowing it to see very small objects such as cells and large molecules. The technique currently employed to solve structure, which was the breakthrough in solving bio-molecule structures by EM, is cryo-electron microscopy ("cryo-em"). The images are created with an electron microscope equipped with a "cold stage". This means that the samples are scanned in a frozen state. This procedure protects them from radiation damage and keeps them in a native state.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structural-bioinformatics/el_basic_concept.png){: loading=lazy }
##### The Advantages of Electron Microscopy

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/advantages-electron-microscopy-6b378294'><i class='fa fa-play' aria-hidden='true'></i></button>The most important advantage of EM is the ability to resolve very large biological complexes. Sometimes EM is the only alternative. One example is the 3-dimensional atomic model of a protein called tubulin that actively participates in cell division. At a resolution of 3.7 &#8491; EM provided the first highly detailed look at tubulin. Structures determined by EM can serve as an initial reference for interpretation of X-ray diffraction patterns. In addition, structural information about transition states and intermediate states in chemical reactions can be collected.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structural-bioinformatics/elec_advantage.png){: loading=lazy }

??? PDB
    **PDB entry: 1tub** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1tub) 
    
##### The Limitations of Electron Microscopy

<button  class='playb' onclick='playAudio(this)' data-mp3-name='structural-bioinformatics/limitations-electron-microscopy-32dcc1a1'><i class='fa fa-play' aria-hidden='true'></i></button>The limitation of EM is the structural resolution that can be achieved. Even though theoretical analysis suggests that there is no problem in achieving high resolutions, technical issues such as optical problems and noise, prevent reaching very high amplifications. For example, the current resolution (about 3.5 &#8491;) is not sufficient to determine the location of the amino-acid side chains.


![](https://media.drugdesign.org/course/structural-bioinformatics/elect_limit.png){: loading=lazy }


Copyright Â© 2024 drugdesign.org
