# Case Studies in Structure-Based Design
!!! Authors
    Claude Cohen (Synergix), Joel van Gelder (InSight Biopharmaceuticals), Dror Shiftan (Pearl Cohen Zedek Latzer LLP)

!!! Info
    Successful examples of structure-based drug design projects are outlined in this chapter with a focus on the drug design features, in particular the analyses that were made and how they were exploited for the successful design of active compounds.


    **Number of Pages: 75 (Â±1 hours read)**

    Last Modified: December 2008

    Prerequisites: Principles of Rational Drug Design, Structure-Based Drug Design: Analysis, Structure-Based Drug Design: Design
## Case Study-1 : Phenyl Imidazoles

### Phenyl-Imidazoles Inhibit Cytochrome P450

Cytochrome P450 is a group of enzymes which contain Heme and catalyze the hydroxylation of aliphatic and aromatic molecules. Camphor is the natural substrate of the bacterial cytochrome P450-cam. Phenylimidazoles, such as phenyl-1, phenyl-2 or phenyl-4, are known to inhibit this hydroxylation process. From a structural point of view a simple rationale could be derived to explain these observations.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/imid_9_2_1_1.png){: loading=lazy }

??? Abstract "articles"
    **Crystal Structures of Metyrapone- and Phenylimidazole-Inhibited Complexes of Cytochrome P-450cam** 
    Poulos TL and Howard AJ 
    Biochemistry 
    26 1987  
    
    **High-Resolution Crystal Structure of Cytochrome P450cam** 
    Poulos TL, Finzel BC, and Howard AJ 
    J Mol Biol. 
    195 1987  
    
### Simple Consideration: Shape Similarity

It is reasonable to assume that the three inhibitors bind to the enzyme in a way where all the phenyl rings are aligned. This superimposition has the advantage of showing that the three molecules have almost identical shapes and can therefore occupy the same hypothetical sub-pocket in the enzyme catalytic site.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/-based-dmedia/structureesign-case-studies/snap_v1_c2_9_2_s2.png){: loading=lazy }

### Perhaps Binding Elements are more Complex ?

However, is the molecular shape the only molecular determinant for binding? Perhaps electrostatic interactions or anchorage with hydrogen bonds are also important! Thus the previous model which considered the geometrical features  alone (and does not distinguish between nitrogen and carbon atoms) is not sufficient. To satisfy the alignment of electronic features, we need to   consider aligning the molecules in such a way that the nitrogen atoms are optimally superimposed. However the resulting model is far from being convincing.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_c2_9_2_s3.gif){: loading=lazy }

### The Structure-Based Answer

In the absence of additional information, we would probably tend to favor the initial pure steric model. In fact additional information is available, as the X-ray structures of the complexes have been solved, including the three phenyl-imidazole inhibitors, the natural substrate camphor and other potent inhibitors. By aligning the proteins of the complexes, the exact solution  how the three phenyl-imidazole analogs should be superimposed is now accessible. One important element in the binding of these molecules is their chelation between one nitrogen atom and the iron of the P450 protein. This is shown in the browser on the next page.
*[analog]: A molecule structurally similar to another, generally based on the same scaffold
*[analogs]: A molecule structurally similar to another, generally based on the same scaffold
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_c2_9_2_s4.gif){: loading=lazy }


??? Abstract "articles"
    **Crystal Structures of Metyrapone- and Phenylimidazole-Inhibited Complexes of Cytochrome P-450cam** 
    Poulos TL and Howard AJ 
    Biochemistry 
    26 1987  
    
### Phenyl-Imidazole Browser


      


![](https://media.drugdesign.org/course/structure-based-design-case-studies/v1_c2_9_3_internet.gif){: loading=lazy }


??? Abstract "articles"
    **Crystal Structures of Metyrapone- and Phenylimidazole-Inhibited Complexes of Cytochrome P-450cam** 
    Poulos TL and Howard AJ 
    Biochemistry 
    26 1987  
    
### Limitations of Chemical Intuition

Although in many cases chemical intuition can compensate for lack of knowledge, there are situations where in the absence essential data, counterproductive models are developed. This example illustrates this situation particularly well and shows the strength of structure-based analyses.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/imid_9_1_0_1.png){: loading=lazy }

## Case Study-2 : BACE-1 Inhibitors
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[Inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.

### BACE-1 Inhibitors
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[Inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.

In order to discover therapeutic agents for treating the etiology of  Alzheimer's disease, Johnson &#38; Johnson addressed the inhibition of the aspartic protease BACE-1 (&beta;-Site APP Cleaving Enzyme), based on the observation that the inhibition of this enzyme lowers &beta;-amyloid levels in the brains of mice. A structure-based strategy was used, which is presented in the following pages.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_target.png){: loading=lazy }

??? Abstract "articles"
    **The novel beta-secretase inhibitor KMI-429 reduces amyloid beta peptide production in amyloid precursor protein transgenic and wild-type mice** 
    Asai M, Hattori C, Iwata N, Saido TC, Sasagawa N, Szabo B, Hashimoto Y, Maruyama K, Tanuma S, Kiso Y, Ishiura S. 
    J Neurochem 
    96(2) 2006 
         
    
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
### Screening the J&#38;J Corporate Compound Collection
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

A random screening of the J&#38;J corporate compound collection led to the identification of a hit such as the 2-amino-3,4-dihydroquinazoline 1, having a Ki of 0.9 &Mu;M. The compound was selected for further studies.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_screening.png){: loading=lazy }

??? Abstract "articles"
    **2-Amino-3,4-dihydroquinazolines as Inhibitors of BACE-1 (beta-Site APP Cleaving Enzyme): Use of Structure Based Design to Convert a Micromolar Hit into a Nanomolar Lead** 
    Ellen W. Baxter, Kelly A. Conway, Ludo Kennis, Francois Bischoff, Marc H. Mercken, Hans L. De Winter, Charles H. Reynolds, Brett A. Tounge, Chi Luo, Malcolm K. Scott, Yifang Huang, Mirielle Braeken, Serge M. A. Pieters, Didier J. C. Berthelot, Stefan Masure, Wouter D. Bruinzeel, Alfonzo D. Jordan, Michael H. Parker, Robert E. Boyd, Junya Qu, Richard S. Alexander, Douglas E. Brenneman, and Allen B. Reitz 
    J.Med.Chem 
    50 2007 
         
    
### Structural Determinants of the Biological Activity of 1

Nothing was known about the structural determinants responsible for the inhibitory activity of compound 1. The J&#38;J group was interested in finding out how the hit compound 1 interacts with BACE-1, to take advantage of such knowledge to rationally develop more potent inhibitors. In the framework of collaboration with Astex Therapeutics, the X-ray structure of BACE-1 complexed with compound 1 was solved.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_determinants.png){: loading=lazy }

??? Abstract "articles"
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
### X-ray Structure of the Complex of 1 with BACE-1

The X-ray structure of the complex of 1 with BACE-1 revealed several interesting features. Molecule 1 was found in an unexpected compact folded conformation, the side chain of the molecule bending back onto itself in a hairpin turn orientation enabling the N-cyclohexyl substituent to occupy the hydrophobic S1 pocket. An exquisite array of hydrogen bonds was formed between the ligand and the two aspartates of the catalytic site (see third button).
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Complex"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_bace_04_b1.png){: loading=lazy }

=== "Ligand alone"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_bace_04_b2.png){: loading=lazy }

=== "H-bonds with aspartates"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_bm1.png){: loading=lazy }

??? Abstract "articles"
    **2-Amino-3,4-dihydroquinazolines as Inhibitors of BACE-1 (beta-Site APP Cleaving Enzyme): Use of Structure Based Design to Convert a Micromolar Hit into a Nanomolar Lead** 
    Ellen W. Baxter, Kelly A. Conway, Ludo Kennis, Francois Bischoff, Marc H. Mercken, Hans L. De Winter, Charles H. Reynolds, Brett A. Tounge, Chi Luo, Malcolm K. Scott, Yifang Huang, Mirielle Braeken, Serge M. A. Pieters, Didier J. C. Berthelot, Stefan Masure, Wouter D. Bruinzeel, Alfonzo D. Jordan, Michael H. Parker, Robert E. Boyd, Junya Qu, Richard S. Alexander, Douglas E. Brenneman, and Allen B. Reitz 
    J.Med.Chem 
    50 2007 
         
    
??? PDB
    **PDB entry: 2Q11** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Q11) 
    
### Flap Flexibility in Aspartyl Proteases

It was also found that the BACE-1 flap (that is located over the active site of the enzyme), was pushed away by a distance of about 4 &#8491; when compared to the structure of the peptidic inhibitor OM99-2. This type of flexibility was also observed for the flap region of other aspartyl proteases such as HIV-1 and Renin.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_flap.png){: loading=lazy }

??? Abstract "articles"
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
??? PDB
    **PDB entry: 2ZHR** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZHR) 
    
### Compound with Increased Folding Capability

The J&#38;J team looked for molecules that mimic the folded conformation of molecule 1 and designed compound 2. The molecule had a longer side-chain and could more easily adopt a hairpin conformation. Also for synthetic reasons the carbonyl of the benzoyl group present in 1 was replaced by an oxygen. This molecule was synthesized and proved to have a K<sub>i</sub> of 158 nM.


=== "2D"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_design2.png){: loading=lazy }
=== "3D superimposition"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_bace_06_b2.png){: loading=lazy }


??? Abstract "articles"
    **2-Amino-3,4-dihydroquinazolines as Inhibitors of BACE-1 (beta-Site APP Cleaving Enzyme): Use of Structure Based Design to Convert a Micromolar Hit into a Nanomolar Lead** 
    Ellen W. Baxter, Kelly A. Conway, Ludo Kennis, Francois Bischoff, Marc H. Mercken, Hans L. De Winter, Charles H. Reynolds, Brett A. Tounge, Chi Luo, Malcolm K. Scott, Yifang Huang, Mirielle Braeken, Serge M. A. Pieters, Didier J. C. Berthelot, Stefan Masure, Wouter D. Bruinzeel, Alfonzo D. Jordan, Michael H. Parker, Robert E. Boyd, Junya Qu, Richard S. Alexander, Douglas E. Brenneman, and Allen B. Reitz 
    J.Med.Chem 
    50 2007 
         
    
### How to Gain Additional Binding

Encouraged by the results obtained on compound 2 and the observation  that the hydrophobic S1' pocket of the enzyme was not occupied by the inhibitors, the J&#38;J team looked for molecules that could form favorable interactions with this pocket. This led to considering substituents having the S stereochemistry on the &alpha;-carbon of the side chain of the inhibitors and helped show that a cyclohexyl group fit perfectly  this pocket perfectly.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_bace_07.png){: loading=lazy }


??? Abstract "articles"
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
### Design of a More Potent Inhibitor
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.

This idea led to the design of 3, which proved to be active at the nanomolar level. The enantiomers of 3 were prepared and  the biological activity was found to reside predominantly in the S enantiomer (3a) as predicted.
*[enantiomer]: Enantiomers are pairs of stereoisomers that are mirror images of one other
*[enantiomers]: Enantiomers are pairs of stereoisomers that are mirror images of one other


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_design3.png){: loading=lazy }

??? Abstract "articles"
    **2-Amino-3,4-dihydroquinazolines as Inhibitors of BACE-1 (beta-Site APP Cleaving Enzyme): Use of Structure Based Design to Convert a Micromolar Hit into a Nanomolar Lead** 
    Ellen W. Baxter, Kelly A. Conway, Ludo Kennis, Francois Bischoff, Marc H. Mercken, Hans L. De Winter, Charles H. Reynolds, Brett A. Tounge, Chi Luo, Malcolm K. Scott, Yifang Huang, Mirielle Braeken, Serge M. A. Pieters, Didier J. C. Berthelot, Stefan Masure, Wouter D. Bruinzeel, Alfonzo D. Jordan, Michael H. Parker, Robert E. Boyd, Junya Qu, Richard S. Alexander, Douglas E. Brenneman, and Allen B. Reitz 
    J.Med.Chem 
    50 2007 
         
    
### X-Ray Structure of the Complex with 3a

The complex of BACE-1 with 3a was solved and confirmed the predicted binding mode of molecule 3a: the occupancy of the S1' pocket by the newly introduced cyclohexyl, a folded conformation of the ligand that put the cyclohexyl group of the side-chain in the S1 pocket, and the network of hydrogen-bond interactions between the N-H groups of the ligand with the two aspartates of the catalytic site of the enzyme.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Complex with 3a"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_bace_09_b1.png){: loading=lazy }

=== "3a and 1 superimposed"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_bace_09_b2.png){: loading=lazy }


??? Abstract "articles"
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
??? PDB
    **PDB entry: 2Q15** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Q15) 
    
### Pharmacological Action of Compound 3a

Compound 3a was further examined to assess its suitability as a lead. The compound showed some selectivity with respect of other aspartic proteases; it exhibited good potency in a cellular assay and proved to lower &beta;-amyloid in rat plasma after p.o. administration.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_.png){: loading=lazy }

??? Abstract "articles"
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
### Important Structural Determinants for Binding

The J&#38;J  group successfully  raised the potency of a weak inhibitor which was found by screening to the nano-molar level. Four important structural determinants were exploited in the design process and enabled the success of this phase of the project: the good anchorage of the 2-aminoquinazoline moiety to the aspartates of the catalytic site; the folded bioactive conformation adopted by the inhibitors enabling it to bind to the hydrophobic S1 pocket and the presence of a yet unexploited hydrophobic pocket in S1'.
*[bioactive conformation]: The geometry adopted by a ligand when it binds to its biological target.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[nce]: New Chemical Entity: a compound not previously described.
*[moiety]: Fragments, functional groups or portions of chemical compounds.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_imp_determinants.png){: loading=lazy }

??? Abstract "articles"
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
### Summary

A structure-based strategy that exploited X-ray crystallography and the docking of new prototypes enabled the rapid discovery of potent inhibitors such as 3a (Ki = 11 nM). The molecule showed good pharmacological properties and is currently being optimized to target Alzheimer's disease.  The synthetic effort would not have been given high priority without the insight provided by the crystal structure of the initial hit bound to BACE-1.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/bace_summary.png){: loading=lazy }

??? Abstract "articles"
    **Emerging beta-amyloid therapies for the treatment of Alzheimer's disease** 
    Conway KA, Baxter EW, Felsenstein KM, Reitz AB 
    Curr Pharm Des 
    9(6) 2003 
         
    
## Case Study-3 : Factor Xa Inhibitors
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[Inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.

### Therapeutic Utility of Factor Xa Inhibitors
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[Inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.

Antithrombotic drugs can be designed by targeting the inhibition of the coagulation cascade outlined below. It was suggested that inhibition of factor Xa would be more efficient than inhibiting thrombin for developing novel antithrombotic agents, on the basis of calculations that show that one molecule of factor Xa generates 138 molecules of thrombin. Here we present  the development of a design strategy that led to the discovery of a new and potent Factor Xa inhibitor.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_1_0_0_1.png){: loading=lazy }

??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### DX-9065a : a Factor Xa Inhibitor
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.

DX-9065a is a potent and  selective factor Xa inhibitor with an IC<sub>50</sub> of 41 nM.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.


=== "2D"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_2_0_0_2.png){: loading=lazy }
=== "3D"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_3_b2.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
??? PDB
    **PDB entry: 1FAX** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FAX) 
    
### Complex Between Factor Xa and DX-9065a
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

The crystal structure of DX-9065a bound to factor Xa has been resolved. It is visualized here with some details.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


=== "Catalytic"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_3_0_1_1.png){: loading=lazy }

=== "Surface"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_3_0_2_1.png){: loading=lazy }

=== "Ribbon"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_3_0_3_1.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Analysis of the Factor Xa and DX-9065a Complex (1/4)

DX-9065a binds to the receptor site with the naphthyl-amidine hydrogen bonding to Asp-189 in the S1 site.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_4_1_0_1.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
??? PDB
    **PDB entry: 1EST** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EST) 
    
#### Analysis of the Factor Xa and DX-9065a Complex (2/4)

The carboxylic acid  makes hydrogen bonds with Gln-192.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_4_2_0_1.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
??? PDB
    **PDB entry: 1EST** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EST) 
    
#### Analysis of the Factor Xa and DX-9065a Complex (3/4)

The acetimiodyl pyrrolidine group binds to the aryl binding pocket S4 in a cation hole created by the carbonyl groups of Lys-96 and Glu-97 together with the Glu-97 side-chain.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_4_3_0_1.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
??? PDB
    **PDB entry: 1EST** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EST) 
    
#### Analysis of the Factor Xa and DX-9065a Complex (4/4)

The interactions of DX-9065a when it binds to factor Xa are represented in the following diagram.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_4_4_0_1.png){: loading=lazy }

??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
??? PDB
    **PDB entry: 1EST** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EST) 
    
### Role of the Carboxylic Acid in Selectivity (1/3)

The specificity of DX-9065a for binding to factor Xa rather than  thrombin is thought to stem for the favorable interaction of the carboxylate with the Gln-192 side-chain of factor Xa.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_10_s1.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
#### Role of the Carboxylic Acid in Selectivity (2/3)

The corresponding residue in thrombin is Glu-192 which is thought to be deprotonated; this would cause a strong repulsive interaction with the carboxylate moiety of DX-9065a.
*[moiety]: Fragments, functional groups or portions of chemical compounds.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_5_2_0_1.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
#### Role of the Carboxylic Acid in Selectivity (3/3)

The following view shows the interactions of DX-9065a with both factor Xa (yellow) and thrombin (blue).


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_10_s3.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Initial Inhibitor Design
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.

The design idea envisioned two phenyl-amidine groups in a molecule with one of them binding in the S1 pocket, and the second in the S4 pocket.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_6_0_0_1.gif){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Design (step 1): Structural Moiety for Pocket S1
*[Moiety]: Fragments, functional groups or portions of chemical compounds.

It was assumed that the first phenyl-amidine group would enter the S1 pocket of factor Xa in a similar way as observed in the complex between trypsin and phenyl-amidine (shown below).
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_1_7_1_0_1.png){: loading=lazy }

#### Phenyl-Amidine Entered into the S1 Pocket 

This phenyl-amidine group was therefore entered into the S1 pocket of factor Xa, in order to form hydrogen bonds with the side-chains of Asp-189 and Gly-218.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_15_s2.png){: loading=lazy }

#### Phenyl-Amidine Oriented in Lowest Energy Orientation

Finally, the lowest energy orientation (in yellow) of the phenyl-amidine group was calculated in the S1 pocket.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_15_s3.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Design (step 2): Structural Moiety for Pocket S4
*[Moiety]: Fragments, functional groups or portions of chemical compounds.

The aryl-binding pocket S4 consists of three aromatic residues: Phe-174, Tyr-99 and Trp-215.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_19_s1.png){: loading=lazy }

#### Phenyl Ring Introduced in Pocket S4

Initially, a phenyl ring was introduced in this pocket (S4) to find an optimal binding orientation.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_19_s2.png){: loading=lazy }

#### Phenyl Substituted with an Amidine 

The phenyl ring fragment was replaced with an amidine group, and the orientation of the phenyl-amidine moiety was then modeled to achieve optimal interactions in the S4 pocket.
*[moiety]: Fragments, functional groups or portions of chemical compounds.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_19_s3.png){: loading=lazy }

#### Stacking Interaction of Phenyl-Amidine with Trp-215

Finally, an orientation was obtained where the amidine stacked  over Trp-215 (3D)


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_19_s4.png){: loading=lazy }

#### Phenyl-Amidine Orientation

Note that the orientation of the phenyl-amidine is different from the position of the DX-9065a acetimidoyl group (in purple).


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_19_s5.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Design (step 3): Design of the Spacer

Assuming the orientation of the phenyl-amidine groups in S1 and in S4 to be preferred, the design process consisted of connecting these fragments without disrupting their respective orientations.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_9_1_0_1.png){: loading=lazy }
#### Phenyl-Amidine Groups in their Preferred Orientations

The following view shows the two phenyl-amidine groups in their preferred orientations.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_25_s2.png){: loading=lazy }

#### Spacer with three Atoms
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer

A 3-atom spacer was found connecting the meta position of the phenyl-amidine located in S1 and the para position of the phenyl-amidine group located in S4.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_9_3_0_1.png){: loading=lazy }

#### Candidate Prototype in the Catalytic Site 

The two phenyl-amidino fragments and the 3-atom spacer are visualized here in the catalytic site of the enzyme.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_25_s4.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Design (step 4): Positioning of the Carboxylate

Finally, the functionalization of the benzylic methylene of the phenyl-amidine group located in pocket S1 proved to be appropriate for introducing a carboxylate group to create favorable interactions with the side-chain of Gln-192, in a similar way as observed with the DX-9065a inhibitor.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_30.png){: loading=lazy }


??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Discovery of a Lead Compound

The initial prototype compound shown below was synthesized and proved to have a good binding affinity for factor Xa with Ki = 34 nM.
*[affinity]: The affinity of a ligand is its ability to bind to its biological target.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_11_0_0_1.png){: loading=lazy }

??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
### Optimization of the Designed Series

The initial lead was further optimized. It was found that introducing a guanidine moiety (compound 21) significantly improved the inhibitory potency for Factor Xa (Ki = 9 nM) with a selectivity of about 10-fold as compared to thrombin.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[moiety]: Fragments, functional groups or portions of chemical compounds.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_12_1_0_1.png){: loading=lazy }
#### Interaction of Compound 21 with Factor Xa

The following is a view of the interaction of compound 21 with factor Xa. Note that this guanidine analog fills the aryl-binding pocket more completely than the amidino group. It also extends deeper into the S4 pocket.
*[analog]: A molecule structurally similar to another, generally based on the same scaffold
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_5_ext_c7_13_32_s2.png){: loading=lazy }

#### Finding an Optimal Spacer

Attempts to extend even deeper into S4 with a 5-atom spacer (compound 23) or more did not lead to better compounds. The preferred distance proved to be close to the 3-atom spacer.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/fxa_1_12_3_0_1.png){: loading=lazy }

??? Abstract "articles"
    **Rational Design and Synthesis of Novel, Potent Bis-phenylamidine Carboxylate Factor Xa Inhibitors** 
    Maduskuie TP Jr., McNamara KJ, Ru Y, Knabb RM and Stouten PFW 
    Journal of Medicinal Chemistry 
    41 1998  
    
## Case Study-4 : Kinase Inhibitors
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[Inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.

### Pyrrolo-Pyrimidine &#38; Quinazoline EGF-R Inhibitors
*[Inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[Inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.

In the process of R&#38;D on  inhibitors of the EGF-R protein kinase as anticancer agents, Novartis discovered the pyrrolo-pyrimidine scaffold while Parke-Davis identified quinazoline structures. In both cases, the molecules proved to be competitive inhibitors of the ATP substrate, although the models used were different. In the following pages, we discuss  the structure-based analyses that were made by the two groups to prove the validity of their respective models. 
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[nce]: New Chemical Entity: a compound not previously described.


=== "2D"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_1_1_1.png){: loading=lazy }
=== "3D"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/snap_v1_c5_8_1_b2.png){: loading=lazy }


??? Abstract "articles"
    **Modelling Study Of Protein Kinase Inhibitors: Binding Mode Of Staurosporine and Origin Of The Selectivity Of CGP 52411** 
    Furet P, Caravatti G, Lydon N, Priestle JP, Sowadski JM, Trinks U and Traxler P 
    J. Comput. Aided. Mol. Des. 
    6 1995  
    
    **Use of a Pharmacophore Model for the Design of EGFr Tyrosine Kinase Inhibitors: 4-(Phenylamino)pyrazolo[3,4,-d]pyrimidines** 
    Trumpp-Kollmeyer S and Showalter HDH 
    Chemtracts - Org. Chem. 
    11 1998  
    
### Novartis and Parke-Davis Opposite Binding Models
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

This account is interesting:  not only because  each team succeeded in finding potent inhibitors in their own series, but also because of the unresolved disagreement. 
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[inhibitors]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_2_0_1.png){: loading=lazy }


??? Abstract "articles"
    **Modelling Study Of Protein Kinase Inhibitors: Binding Mode Of Staurosporine and Origin Of The Selectivity Of CGP 52411** 
    Furet P, Caravatti G, Lydon N, Priestle JP, Sowadski JM, Trinks U and Traxler P 
    J. Comput. Aided. Mol. Des. 
    6 1995  
    
    **Use of a Pharmacophore Model for the Design of EGFr Tyrosine Kinase Inhibitors: 4-(Phenylamino)pyrazolo[3,4,-d]pyrimidines** 
    Trumpp-Kollmeyer S and Showalter HDH 
    Chemtracts - Org. Chem. 
    11 1998  
    
### Controversy: Novartis &#38; Parke-Davis Binding Modes
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[Binding Modes]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

The disagreement relates to how the molecules are recognized by the protein. The view shown here summarizes the binding mode imagined by each group for their own molecules.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_3_1_1.png){: loading=lazy }

#### Parke-Davis Analyses

The following view illustrates the binding mode used by Parke-Davis for its own series (left) and how they viewed the binding of the Novartis molecules (right).  
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_3_2_1.png){: loading=lazy }

#### Novartis Analyses

The following view illustrates the binding mode used by Novartis for its own series. It was shown on the previous page that two alternative binding modes can be imagined for the Novartis molecule, however  only one binding mode can apparently be modeled for the Parke-Davis compound. 
                
               
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[binding modes]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_3_3_1.png){: loading=lazy }


??? Abstract "articles"
    **Modelling Study Of Protein Kinase Inhibitors: Binding Mode Of Staurosporine and Origin Of The Selectivity Of CGP 52411** 
    Furet P, Caravatti G, Lydon N, Priestle JP, Sowadski JM, Trinks U and Traxler P 
    J. Comput. Aided. Mol. Des. 
    6 1995  
    
    **Use of a Pharmacophore Model for the Design of EGFr Tyrosine Kinase Inhibitors: 4-(Phenylamino)pyrazolo[3,4,-d]pyrimidines** 
    Trumpp-Kollmeyer S and Showalter HDH 
    Chemtracts - Org. Chem. 
    11 1998  
    
### X-ray Structure of ATP Bound to a Kinase 

### The Rational Drug Design Strategies

Lead compounds were discovered through pharmacophore-based strategies. However, the crystal structure of a cyclic-AMP dependent kinase complexed with ATP, the cofactor of the protein kinases, revealed the key hydrogen bonds exploited by ATP in its binding to the kinase. This first crystallographic structure opened the door to a structure-based design perspective and facilitated the optimization of the leads.
*[pharmacophore]: Specific 3D arrangement of chemical groups common to active molecules and essential to their biological activities.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_4_0_1.png){: loading=lazy }

??? Abstract "articles"
    **Crystal Structure of the Catalytic Subunit of cAMP-Dependent Protein Kinase Complexed with MgATP and Peptide Inhibitor** 
    Zheng J, Knighton DR, ten Eyck LF, Karlsson R, Xuong N, Taylor SS, and Sowadski JM 
    Biochemistry 
    32 1993  
    
??? PDB
    **PDB entry: 1ATP** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ATP) 
    
### Binding Mode of ATP
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

ATP, the cofactor of the protein kinases was found to anchor to the kinases in a bidentate manner with backbone atoms of two residues of the enzyme. This anchorage, reminiscent of the Watson-Crick mode of base pairing of adenine nucleotides, appeared to be a key determinant in the recognition of ATP by the kinases.
*[bidentate]: Having two anchorage points


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_5_1_1.png){: loading=lazy }

### Binding Mode of Staurosporine
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

Staurosporine is an ATP competitive potent protein kinase inhibitor. Despite its seemingly unrelated chemical structure it was shown that the compound could exploit the same hydrogen bond interactions as ATP.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_5_2_1.png){: loading=lazy }


??? Abstract "articles"
    **Modelling Study Of Protein Kinase Inhibitors: Binding Mode Of Staurosporine and Origin Of The Selectivity Of CGP 52411** 
    Furet P, Caravatti G, Lydon N, Priestle JP, Sowadski JM, Trinks U and Traxler P 
    J. Comput. Aided. Mol. Des. 
    6 1995  
    
### Homology Model of EGF-R Catalytic Site

The X-ray data also served in  the construction of a model of the EGF-R kinase by homology and the identification of different areas in the active site (the anchorage, the "sugar" pocket, and the "large" pocket).


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_5_3_1.png){: loading=lazy }
### From Staurosporine to Pyrrolo-pyrimidine

The Novartis predicted binding mode for Staurosporine proved to be correct as demonstrated by subsequent X-ray studies of complexes with other protein kinases. Novartis ascribed a binding mode exploiting the same hydrogen bond interactions to the pyrrolo-pyrimidine series.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_6_0_1.png){: loading=lazy }


??? Abstract "articles"
    **Modelling Study Of Protein Kinase Inhibitors: Binding Mode Of Staurosporine and Origin Of The Selectivity Of CGP 52411** 
    Furet P, Caravatti G, Lydon N, Priestle JP, Sowadski JM, Trinks U and Traxler P 
    J. Comput. Aided. Mol. Des. 
    6 1995  
    
### The Novartis Binding Mode of Pyrrolo-pyrimidine
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

The Novartis group placed the anilino moiety in the relative small region defined as the "sugar" pocket because the biological activities were  very sensitive to the substitution in the phenyl ring. For example only a para-fluoro substitution is accepted as opposed to all other analogs with a substitutions in para, (e.g. with a chlorine, a methyl or ethyl) which were inactive.
*[analog]: A molecule structurally similar to another, generally based on the same scaffold
*[analogs]: A molecule structurally similar to another, generally based on the same scaffold
*[moiety]: Fragments, functional groups or portions of chemical compounds.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_7_1_1.png){: loading=lazy }

??? Abstract "articles"
    **Modelling Study Of Protein Kinase Inhibitors: Binding Mode Of Staurosporine and Origin Of The Selectivity Of CGP 52411** 
    Furet P, Caravatti G, Lydon N, Priestle JP, Sowadski JM, Trinks U and Traxler P 
    J. Comput. Aided. Mol. Des. 
    6 1995  
    
#### The Pyrrole Ring in the Large Pocket

The pyrrole ring was placed in the "large" pocket area because substitutions of the heterocycle by many hydrophobic groups improved the biological activities. The analogs with two methyls, one methyl and one phenyl or two additional phenyl rings substantially increased  biological activities as shown below.
*[analog]: A molecule structurally similar to another, generally based on the same scaffold
*[analogs]: A molecule structurally similar to another, generally based on the same scaffold


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_7_2_1.png){: loading=lazy }
#### The Pyrrole Ring Pointing Towards the Sugar Pocket

The "reverse" binding mode, with the pyrrole ring pointing towards the sugar pocket, is not a valid model. It eliminates the possibility of substitutions on the pyrrole ring. Such analogs were actually synthesized and increased the biological activities, which could not be accounted for by this model.
*[analog]: A molecule structurally similar to another, generally based on the same scaffold
*[analogs]: A molecule structurally similar to another, generally based on the same scaffold
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_7_3_1.png){: loading=lazy }
### Parke-Davis Analyses the Quinazoline Scaffold

The rationale behind the Parke-Davis group's idea for modeling the binding mode of their quinazoline scaffold was based on systematic exploration of the anchorage capabilities of this scaffold. The two nitrogens of the quinazoline nucleus were shown to be essential for these  activities (inactive compounds are obtained when any of the nitrogens is replaced by a carbon). The Parke-Davis group concluded that both atoms should be used as hydrogen bond acceptors in the interaction with the enzyme.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_8_1_1.png){: loading=lazy }
#### Additional SAR Analyses made by Parke-Davis
*[SAR]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.

Substitution of the phenyl ring of the anilino side chain greatly reduced activities indicating limited bulk tolerance, and only the analog with bromine in meta showed improved activities.
*[analog]: A molecule structurally similar to another, generally based on the same scaffold
*[nce]: New Chemical Entity: a compound not previously described.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_8_2_1.png){: loading=lazy }

??? Abstract "articles"
    **Use of a Pharmacophore Model for the Design of EGFr Tyrosine Kinase Inhibitors: 4-(Phenylamino)pyrazolo[3,4,-d]pyrimidines** 
    Trumpp-Kollmeyer S and Showalter HDH 
    Chemtracts - Org. Chem. 
    11 1998  
    
### Parke-Davis Model of the Quinazoline Analogs
*[Analog]: A molecule structurally similar to another, generally based on the same scaffold
*[Analogs]: A molecule structurally similar to another, generally based on the same scaffold

All the elements assembled by the Parke-Davis group lead to  considering the following binding mode for the quinazoline series.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_9_1_1.png){: loading=lazy }
#### Specificity Observed in EGF-R Kinase Inhibition

An additional element that supported this model was the specificity observed in EGF-R activities. This can be explained by favorable hydrophobic interactions with Cys-751 that exist only in the EGF-R enzyme.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_9_2_1.png){: loading=lazy }
#### Anilino Towards the Sugar Pocket not Reasonable

The "reverse" binding mode, with the anilino side chain pointing towards the sugar pocket, is a model that does not explain the observed selectivity.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_9_3_1.png){: loading=lazy }

??? Abstract "articles"
    **Use of a Pharmacophore Model for the Design of EGFr Tyrosine Kinase Inhibitors: 4-(Phenylamino)pyrazolo[3,4,-d]pyrimidines** 
    Trumpp-Kollmeyer S and Showalter HDH 
    Chemtracts - Org. Chem. 
    11 1998  
    
### Parke-Davis Model Consistent with Observed SAR
*[SAR]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.

This binding mode indicates that the available room around the nitrogen area is very limited, whereas a third ring can be fused to the bicyclic quinazoline ring without loss of binding affinity. Docking studies showed that the binding mode represented here satisfied all the observed SAR.
*[affinity]: The affinity of a ligand is its ability to bind to its biological target.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[SAR]: Structure-Activity Relationships (SAR) is the analysis of the relationships between chemical structure and biological activity.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_10_0_1.png){: loading=lazy }

??? Abstract "articles"
    **Use of a Pharmacophore Model for the Design of EGFr Tyrosine Kinase Inhibitors: 4-(Phenylamino)pyrazolo[3,4,-d]pyrimidines** 
    Trumpp-Kollmeyer S and Showalter HDH 
    Chemtracts - Org. Chem. 
    11 1998  
    
### Binding Mode of the Pyrrolo-Pyrimidine Series
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

The following example illustrates the binding mode of  pyrrolo-pyrimidine as imagined by Novartis or Parke-Davis.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_11_1_1.png){: loading=lazy }


??? Abstract "articles"
    **Use of a Pharmacophore Model for the Design of EGFr Tyrosine Kinase Inhibitors: 4-(Phenylamino)pyrazolo[3,4,-d]pyrimidines** 
    Trumpp-Kollmeyer S and Showalter HDH 
    Chemtracts - Org. Chem. 
    11 1998  
    
### Binding Mode of the Quinazoline Series
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

The following example illustrates the binding mode of the quinazoline as imagined by Novartis or Parke-Davis.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_11_2_1.png){: loading=lazy }
=== "Parke-Davis"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_11_2_1_f1.png){: loading=lazy }
=== "Novartis"
    ![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_11_2_1_f2.png){: loading=lazy }
### What is the Correct Solution?

The dispute is not yet resolved. In principle, there are three possibilities.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_11_3_1.png){: loading=lazy }
### Ligand Observed with a Novartis Binding Mode
*[Ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

The following example illustrates the X-ray structure of a complex of NU2058 with another protein kinase, CDK2, revealing an orientation of the ligand in a Novartis-like binding mode.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_12_1_1.png){: loading=lazy }


??? Abstract "articles"
    **Identification of Novel Purine and Pyrimidine Cyclin-Dependent Kinase Inhibitors with Distinct Molecular Interactions and Tumor Cell Growth Inhibition Profiles** 
    Arris CE, Boyle FT, Calvert AH, Curtin NJ, Endicott JA, Garman EF, Gibson AE, Golding BT, Grant S, Griffin RJ, Jewsbury P, Johnson LN, Lawrie AM, Newell DR, Noble ME, Sausville EA, Schultz R and Yu W 
    J. Med. Chem. 
    43 2000  
    
??? PDB
    **PDB entry: 1E1V** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1E1V) 
    
#### Alignment with the Novartis Model

The Novartis model (yellow) is displayed aligned with the X-ray structure of the NU2058 bound to CDK2.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_12_2_1.png){: loading=lazy }

### Ligand Observed with a Parke-Davis Binding Mode
*[Ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[Binding Mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.

The following example illustrates the X-ray structure of a complex of a 4-anilinoquinazoline derivative with the protein kinase p38, revealing an orientation of the ligand in a Parke-Davis-like binding mode.
*[ligand]: In Biochemistry: substance that binds to a biological receptor. In Chemistry: an atom or group of atoms.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_13_1_1.png){: loading=lazy }


??? Abstract "articles"
    **Binding Mode of the 4-Anilinoquinazoline Class of Protein Kinase Inhibitor: X-ray Crystallographic Studies of 4-Anilinoquinazolines Bound to Cyclin-Dependent Kinase 2 and p38 Kinase** 
    Shewchuk L, Hassell A, Wisely B, Rocque W, Holmes W, Veal J and Kuyper LF 
    J. Med. Chem. 
    43 2000  
    
    **Structure of the Epidermal Growth Factor Receptor Kinase Domain Alone and in Complex with a 4-Anilinoquinazoline Inhibitor** 
    Stamos J, Sliwkowski MX, Eigenbrot C 
    J Biol Chem. 
    277 2002  
    
??? PDB
    **PDB entry: 1DI9** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DI9) 
    
    **PDB entry: 1M17** 
     
    [](http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M17) 
    
#### Alignment with the Parke-Davis Model

The Parke-Davis model (yellow) is displayed aligned with the X-ray structure of the 4-anilinoquinazoline bound to p38.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_13_2_1.png){: loading=lazy }

#### X-Ray Resolution of Tarceva Bound to EGF-R Kinase

Very recently, the crystal structure of the EGF-R kinase domain complexed with Tarceva (an inhibitor currently in development) has been determined. The orientation of the inhibitor in the complex corresponds to the Parke-Davis model and confirms the existence of a water molecule, as already observed in p38-inhibitor complexes.
*[inhibitor]: Chemical substance that blocks or suppress the activity of a given enzyme.
*[nce]: New Chemical Entity: a compound not previously described.
*[ee]: stands for "enantiomeric excess". A quantity for measuring the optical purity and represents the percentage of the major enantiomer minus the percentage of the minor enantiomer
*[rms]: Root Mean Square: the square root of the arithmetic average of the square's set of values.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/alt1_8_13_3_1.png){: loading=lazy }

### Conclusion

Two opposite models were derived from analyses of activities on the EGF-R kinase. Recent X-ray studies of closely related structures complexed with other kinases indicated that both binding modes could exist. This example showed that one must be prepared to consider more than one binding mode. This diversity created some complications in the analyses but also opened up a broader range of possibilities for the design of new structures.
*[binding mode]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.
*[binding modes]: Orientation and geometry adopted by a chemical substance when it is bound to its receptor.


![](https://media.drugdesign.org/course/structure-based-design-case-studies/egft_8_14_0_1.png){: loading=lazy }

## ADDITIONAL CASE STUDIES

### Additional Case Studies


      


![](https://media.drugdesign.org/course/structure-based-design-case-studies/sbdd_case_other_examples.png){: loading=lazy }

Copyright Â© 2022 drugdesign.org